%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Masters/Doctoral Thesis 
% LaTeX Template
% Version 2.4 (22/11/16)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Version 2.x major modifications by:
% Vel (vel@latextemplates.com)
%
% This template is based on a template by:
% Steve Gunn (http://users.ecs.soton.ac.uk/srg/softwaretools/document/templates/)
% Sunil Patel (http://www.sunilpatel.co.uk/thesis-template/)
%
% Template license:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[
12pt, % The default document font size, options: 10pt, 11pt, 12pt
%oneside, % Two side (alternating margins) for binding by default, uncomment to switch to one side
english, % ngerman for German
singlespacing, % Single line spacing, alternatives: onehalfspacing or doublespacing
%draft, % Uncomment to enable draft mode (no pictures, no links, overfull hboxes indicated)
%nolistspacing, % If the document is onehalfspacing or doublespacing, uncomment this to set spacing in lists to single
%liststotoc, % Uncomment to add the list of figures/tables/etc to the table of contents
%toctotoc, % Uncomment to add the main table of contents to the table of contents
%parskip, % Uncomment to add space between paragraphs
%nohyperref, % Uncomment to not load the hyperref package
headsepline, % Uncomment to get a line under the header
chapterinoneline, % Uncomment to place the chapter title next to the number on one line
%consistentlayout, % Uncomment to change the layout of the declaration, abstract and acknowledgements pages to match the default layout
]{MastersDoctoralThesis} % The class file specifying the document structure

\usepackage[utf8]{inputenc} % Required for inputting international characters
\usepackage[T1]{fontenc} % Output font encoding for international characters

\usepackage{palatino} % Use the Palatino font by default

\usepackage[backend=bibtex,style=authoryear,natbib=true]{biblatex} % Use the bibtex backend with the authoryear citation style (which resembles APA)

\addbibresource{thesis-articles.bib} % The filename of the bibliography

\usepackage[autostyle=true]{csquotes} % Required to generate language-dependent quotes in the bibliography

% Elliott-added packages
\usepackage{amsmath}
\usepackage{listings}
\usepackage{float}

%----------------------------------------------------------------------------------------
%	MARGIN SETTINGS
%----------------------------------------------------------------------------------------

\geometry{
	paper=a4paper, % Change to letterpaper for US letter
	inner=2.5cm, % Inner margin
	outer=2.5cm, % Outer margin
	bindingoffset=.5cm, % Binding offset
	top=1.5cm, % Top margin
	bottom=1.5cm, % Bottom margin
	%showframe, % Uncomment to show how the type block is set on the page
}

%----------------------------------------------------------------------------------------
%	THESIS INFORMATION
%----------------------------------------------------------------------------------------

\thesistitle{Simulating dynein'’s powerstroke using Brownian dynamics} % Your thesis title, this is used in the title and abstract, print it elsewhere with \ttitle
\supervisor{Dr. David \textsc{Roundy}} % Your supervisor's name, this is used in the title page, print it elsewhere with \supname
\examiner{} % Your examiner's name, this is not currently used anywhere in the template, print it elsewhere with \examname
\degree{Bachelor of Science} % Your degree name, this is used in the title page and abstract, print it elsewhere with \degreename
\author{Elliott \textsc{Capek}} % Your name, this is used in the title page and abstract, print it elsewhere with \authorname
\addresses{} % Your address, this is not currently used anywhere in the template, print it elsewhere with \addressname

\subject{Physics} % Your subject area, this is not currently used anywhere in the template, print it elsewhere with \subjectname
\keywords{} % Keywords for your thesis, this is not currently used anywhere in the template, print it elsewhere with \keywordnames
\university{\href{http://www.university.com}{Oregon State University}} % Your university's name and URL, this is used in the title page and abstract, print it elsewhere with \univname
\department{\href{http://physics.oregonstate.edu}{Physics}} % Your department's name and URL, this is used in the title page and abstract, print it elsewhere with \deptname
\group{\href{http://physics.oregonstate.edu/~roundyd/}{Roundy Research Group}} % Your research group's name and URL, this is used in the title page, print it elsewhere with \groupname
\faculty{} % Your faculty's name and URL, this is used in the title page and abstract, print it elsewhere with \facname

\AtBeginDocument{
\hypersetup{pdftitle=\ttitle} % Set the PDF's title to your title
\hypersetup{pdfauthor=\authorname} % Set the PDF's author to your name
\hypersetup{pdfkeywords=\keywordnames} % Set the PDF's keywords to your keywords
}

\begin{document}

\frontmatter % Use roman page numbering style (i, ii, iii, iv...) for the pre-content pages

\pagestyle{plain} % Default to the plain heading style until the thesis style is called for the body content

%----------------------------------------------------------------------------------------
%	TITLE PAGE
%----------------------------------------------------------------------------------------

\begin{titlepage}
\begin{center}

\vspace*{.06\textheight}
{\scshape\LARGE \univname\par}\vspace{1.5cm} % University name
\textsc{\Large Undergraduate Thesis}\\[0.5cm] % Thesis type

\HRule \\[0.4cm] % Horizontal line
{\huge \bfseries \ttitle\par}\vspace{0.4cm} % Thesis title
\HRule \\[1.5cm] % Horizontal line
 
\begin{minipage}[t]{0.4\textwidth}
\begin{flushleft} \large
\emph{Author:}\\
{\authorname} % Author name - remove the \href bracket to remove the link
\end{flushleft}
\end{minipage}
\begin{minipage}[t]{0.4\textwidth}
\begin{flushright} \large
\emph{Supervisor:} \\
{\supname} % Supervisor name - remove the \href bracket to remove the link  
\end{flushright}
\end{minipage}\\[3cm]
 
\vfill

\large \textit{A thesis submitted in fulfillment of the requirements\\ for the degree of \degreename}\\[0.3cm] % University requirement text
\textit{in the}\\[0.4cm]
\groupname\\\deptname\\[2cm] % Research group name and department name
 
\vfill

{\large \today}\\[4cm] % Date
%\includegraphics{Logo} % University/department logo - uncomment to place it
 
\vfill
\end{center}
\end{titlepage}

%----------------------------------------------------------------------------------------
%	ABSTRACT PAGE
%----------------------------------------------------------------------------------------

\begin{abstract}
\addchaptertocentry{\abstractname} % Add the abstract to the table of contents
Dynein is a motor protein which transports cargo along tracks inside the cell. Like related motor proteins kinesin and myosin, dynein uses cellular energy to take steps with its two foot domains. Unlike kinesin or myosin, dynein's stepping pattern is highly varied: it can take steps between zero and 60nm in both the forwards and backwards directions. It is believed that dynein takes such broad, stochastic steps because its large size and several elastic regions make it more influenced by Brownian motion. To test this, we model the motor as a 2D system of elastic hinges, then simulate this model using Brownian dynamics. Preliminary results indicate such a model is capable of taking steps between zero and 25nm. These results give hope that, with further tweaking, the model may be able to generate both larger steps and backwards steps.
\end{abstract}

%% %----------------------------------------------------------------------------------------
%% %	ACKNOWLEDGEMENTS
%% %----------------------------------------------------------------------------------------

%% \begin{acknowledgements}
%% \addchaptertocentry{\acknowledgementname} % Add the acknowledgements to the table of contents
%% The acknowledgments and the people to thank go here, don't forget to include your project advisor\ldots
%% \end{acknowledgements}

%----------------------------------------------------------------------------------------
%	LIST OF CONTENTS/FIGURES/TABLES PAGES
%----------------------------------------------------------------------------------------

\tableofcontents % Prints the main table of contents

\listoffigures % Prints the list of figures

\listoftables % Prints the list of tables

%----------------------------------------------------------------------------------------
%	ABBREVIATIONS
%----------------------------------------------------------------------------------------

\begin{abbreviations}{ll} % Include a list of abbreviations (a table of two columns)

\textbf{MT} & \textbf{M}icro\textbf{t}ubule\\
\textbf{MTBD} & \textbf{M}icro\textbf{t}ubule \textbf{B}inding \textbf{D}omain\\

\end{abbreviations}

%% %----------------------------------------------------------------------------------------
%% %	PHYSICAL CONSTANTS/OTHER DEFINITIONS
%% %----------------------------------------------------------------------------------------

%% \begin{constants}{lr@{${}={}$}l} % The list of physical constants is a three column table

%% % The \SI{}{} command is provided by the siunitx package, see its documentation for instructions on how to use it

%% Speed of Light & $c_{0}$ & \SI{2.99792458e8}{\meter\per\second} (exact)\\
%% %Constant Name & $Symbol$ & $Constant Value$ with units\\

%% \end{constants}

%% %----------------------------------------------------------------------------------------
%% %	SYMBOLS
%% %----------------------------------------------------------------------------------------

%% \begin{symbols}{lll} % Include a list of Symbols (a three column table)

%% $a$ & distance & \si{\meter} \\
%% $P$ & power & \si{\watt} (\si{\joule\per\second}) \\
%% %Symbol & Name & Unit \\

%% \addlinespace % Gap to separate the Roman symbols from the Greek

%% $\omega$ & angular frequency & \si{\radian} \\

%% \end{symbols}

%% %----------------------------------------------------------------------------------------
%% %	DEDICATION
%% %----------------------------------------------------------------------------------------

%% \dedicatory{For/Dedicated to/To my\ldots} 

%----------------------------------------------------------------------------------------
%	THESIS CONTENT - CHAPTERS
%----------------------------------------------------------------------------------------

\mainmatter % Begin numeric (1,2,3...) page numbering

\pagestyle{thesis} % Return the page headers back to the "thesis" style

% Include the chapters of the thesis as separate files from the Chapters folder
% Uncomment the lines as you write the chapters

%----------------------------------------------------------------------------------------
%	THESIS CONTENT - CHAPTERS
%----------------------------------------------------------------------------------------
\chapter{Introduction}
\label{chap:Introduction}

\blockquote{An embryo in its spherical blastula stage has spherical symmetry [...]. But a system which has spherical symmetry, and whose state is changing because of chemical reactions and diffusion, will remain spherically symmetrical for ever [...]. It certainly cannot result in an organism such as a horse, which is not spherically symmetrical. \textbf{Alan Turing, The Chemical Basis of Morphogenesis, 1952}}

Cells need to create asymmetric, heterogeneous structures inside themselves in order to function. However, diffusive processes cannot create this asymmetry. How then do cells create the heterogeneous structures they need to exist? Motor proteins are part of the answer. Motor proteins are structures which use energy to create directed motion in cells. In broad terms, they are mechanical creators of asymmetry. This thesis explores a hypothesis about how one such motor protein, dynein, generates directed motion.\\

%% \section{Interest builder}
%% Begin with a paper-like introduction which talks about:

%% \begin{itemize}
%% \item How dynein is a structure which turns chemical energy into motion
%% \item How dynein's variation in step size and directionality suggest a large diffusion component \cite{reck2006single}
%% \item How it's weird that a cellular structure can generate directed motion
%% \item Other cool stuff - make dynein mysterious and cool.
%% \end{itemize}

%% Tell a story about dynein! Present the project as a story -- we don't have knowledge abotu dynein, it's really weird, we want to get info about it

\section{Motivation}
Motor proteins are cellular structures which use chemical energy to do work and generate motion. Kinesin, myosin and dynein are the three main motor proteins in cells. Though these proteins have significantly different ancestors \ref{cite}, they all evolved into a single similar structure. In biology when a structure is evolved three separate times, it is a sign that nature has found an efficient way of solving a problem. How, then, do motor proteins work?\\

The basic process is simple. Kinesin, myosin and dynein each use two binding domains to attach to a surface. These binding domains are connected via a linker domain. Each motor protein is an engine which undergoes cycles, where each cycle moves the motor across a surface. In a single cycle a binding domain unbinds from a surface, tenses its foot to kick forward, rebinds, then untenses its foot to pull the whole motor forward. This pattern is repeated by both feet to generate motion \cite{kinesin-cycle} \cite{myosin-cycle}.\\

Although the three motor protein families operate using the same stepping cycle, their behaviors differ significantly. Kinesin and myosin are predictable motors which step in the forward direction with a regular step size [get kinesin, myosin stepping papers]. In contrast, dynein's stepping pattern varies in both size and direction. The motor can take 64nm steps forward or 32nm steps backward, and often steps sideways as well \cite{weihongpaper}. How can motors which implement the same release, lever, bind, lurch stepping mechanism behave so differently?\\

One possibility is that dynein's size and organization are responsible for the motor's odd stepping pattern. Dynein is much larger than kinesin or myosin, so its feet may need to move significantly further during each cycle. Due to the random nature of Brownian motion from water particle collisions, there may be significantly more variance in the path taken by dynein's feet compared to other motor feet. This could explain the large variance in step size. Another contributing factor could be that dynein is so large that it can twist and contort during its step, allowing varying and even backwards steps, which a small motor would be too stiff to achieve.\\

This thesis describes an experiment which aims to test whether Brownian motion and flexibility cause dynein's unique stepping pattern. To do this, we construct a mathematical model of the dynein motor which exhibits flexibility and feels Brownian forces. We then simulate the model and compare its behavior to real dynein.\\

%% Dynein:
%% feet which bind to track
%% motor connected to feet which generates the force
%% -----------
%% ATP binding causes dissociation
%% Also causes forward kick
%% ATP hydrolysis
%% rebinding causes ADP rejection and forward lurch
%% restart

%% Kinesin:
%% feet which bind to track
%% connected via lever arm to cargo binding domain
%% -----------
%% ATP binding causes forward kick
%% ATP hydrolysis
%% MT binding, ADP, Pi rejection
%% Other foot unbinds

%% Myosin:
%% Feet which bind to track
%% connected via lever arm to cargo binding domain
%% ----------
%% actin binding causes Pi rejection, strengthening binding
%% binding causes foot to change relative to lever, shifting the motor forward
%% ATP binding causes dissocaition and undoing the powerstroke

\section{What is dynein?}
\subsection{Basics}
Dynein is a cellular structure which converts chemical energy to mechanical energy. It does so by reacting with adenosine triphosphate (ATP) to take steps along long cellular tracks known as microtubules (MTs). Dynein's mechanical energy is used for various cellular functions, including to transport molecules around the cell, pull chromosomes during division, and move cellular propellers known as cilia \cite{cianfroccoreview}.\\

Dynein is a protein with many smaller domains. A depiction of the protein and its labeled subdomains is shown in Figure (\ref{dynein-artist-rendition}). Each motor is a homodimer made up of two identical monomer proteins. Each monomer has a binding domain which binds to the microtubule, a motor domain which binds ATP, a tail domain which connects to the other monomer, a stalk which connects the binding and motor domains, and a linker which connects the motor and tail domains. The tail domain is also responsible for binding to cargo.\\

\begin{figure}[h]
  \centering
  \includegraphics[width=.65\textwidth,keepaspectratio]{../../figures/dynein-artist-rendition.jpg}
  \caption{Artist's rendition of dynein bound to a microtubule.}{One monomer is bound to the MT, and the other is raised up. Modified. Source: Lander Lab, The Scripps Research Institute.}
  \label{dynein-artist-rendition}
\end{figure}

Dynein is believed to move by using the energy of ATP to cycle through a ring of states, each with different spatial relations to the microtubule. By cycling forward through this ring, the molecule will end up in the same state it began, but moved forward a small amount. This ring of states is known as the mechanochemical cycle \cite{cianfroccoreview}. The goal of this project is to create a mathematical model of the mechanochemical cycle and verify that it can reproduce experimental dynein stepping data. There are several types of dynein which all behave differently. This project will focus on cytoplasmic dynein-1, hereby referred to as dynein.\\

\subsection{Why do cells need motor transport?}
Cells are organized, heretogenous structures which respond quickly to their environments. This means that cells require a mechanism for rapidly moving components to precise locations within the cell. This can be a challenge, since cells are fairly large compared to the proteins which compose them. A human fibroblast cell has a volume of roughly 2000 $\mu m^3$\cite{fibroblastvolume}, corresponding to roughly $8 \mu m$ in diameter. In comparison, human hemoglobin has a diameter of roughly $5 nm$ (PDB id 5ME2), a $10^3$ factor difference in size.\\

Diffusion, or random motion of molecules due to collisions with solvent (i.e. Brownian motion), is one possible process cells could use to transport biomolecules. Diffusion has two problems: it is slow and nondirected. The expected distance covered for one-dimensional Brownian motion after time t was found by Einstein to be \cite{einstein-paper}:

\begin{equation}
  \label{diffusion-equation}
  <x> = \sqrt{2Dt}
\end{equation}

where D is the diffusion constant describing the diffusability of the biomolecule. For a medium-sized (140kD, 1D = weight of H atom) protein with a diffusion constant $D = .2 \mu m^2/s$ \cite{diffusionconstants}, it would take about a month to travel across a millimeter-sized cell. In contrast, it would take dynein, which travels at roughly 100nm/s \cite{weihongpaper}, a much more reasonable three hours to do the same. Figure (\ref{fig:diffusion_vs_dynein}) demonstrates how motor transport quickly outstrips diffusion in a cell.\\

\begin{figure}[h]
  \centering
  \includegraphics[width=.65\textwidth,keepaspectratio]{../../figures/diffusion_vs_dynein.png}
  \caption{Passive diffusion vs motor transport rate}{Plot comparing the diffusion rate of a 140kD protein with a diffusion consant $D = .2 \mu m^2/s$ with a dynein motor travelling at 100nm/s.}
  \label{fig:diffusion_vs_dynein}
\end{figure}

\textit{Maybe use the more reasonable 2$\mu m / s$ number for dynein velocity, in-vitro speed may not be applicable since [ATP]-limited}

Another advantage of motor transport is that microtubules can be highly specific in their location in the cell. During cell division chromosomes line up at the center of each cell in a plane. Such a geometrically precise shape requires some sort of directed motion which a random process could never provide.\\

\textit{maybe decorate the above plot with pictures of cells of each size, or how long certain cellular events take? That could be cool and make it actually an interesting plot}
\textit{this argument would be much better if you could find examples of things which need to be done quickly in the cell, like responding to foreign particles or signalling molecules or cell division or neural events}

\subsection{What role does dynein play in the cell?}
Some of dynein's primary roles in the cell are to transport cellular cargos such as organelles, vesicles, mRNA, cytoskeletal filaments and certain proteins. Dynein also plays a role in positioning and breaking down the nucleus, cell death, spindle formation and the placement of other important cellular structures \cite{valetoolbox}.\\

Microtubules are long polymers of alternating $\alpha-$ and $\beta-$ tubulin subunits. MTs are polar, meaning they have a distinct directionality based on the orientation of thetubulin proteins which comprise them. One pole, the minus end, is where they initiate formation, typically at a MTOC (microtubule organizing center) around the center of the cell. The other pole, the plus end, is where they grow and shrink from. Kinesin and myosin, two other families of motor protein, typically walk towards the plus end of the MT. Dynein is unique in that it walks towards the minus end, making it a minus end-directed motor. This lends it to a particularly important function inside neurons known as retrograde transport.\\

Neurons have cell bodies at their centers and axons which grow outwards. Axons are long, narrow structures extending up to a meter in length in humans. Cell bodies contain nuclei and important organelles for synthesizing proteins, but growth of axon tips is vital for neuron function. This means bidirectional motor transport between axon tips and cell bodies is very important in neurons, since diffusion would not be quick enough. Retrograde transport is transport from the axon tip to the cell body. Cargo includes vesicles full of proteins ready to be broken down in the cell body and microtubule fragments to be returned from the axon tip \cite{neuroanatomy}. Because microtubule minus ends are found in the cell body, dynein's minus end-directed nature makes it the only protein capable of retrograde transport. An interesting question is, if proteins like dynein are synthesized in the cell body but needed at the axon tip to perform retrograde transport, how do dynein get to the tip? The answer is shown in \ref{retrograde-transport}: kinesin, a plus end-directed motor, takes dynein to the tip from the cell body.\\

\begin{figure}[h]
  \centering
  \includegraphics[width=.65\textwidth,keepaspectratio]{../../figures/retrograde_transport.jpg}
  \caption{Dynein localization to axon growth cones.}{Mechanism of dynein localization to axon tips, or growth cones, in retrograde transport mediated by kinesin anterograde transport. Modified from \cite{valetoolbox}.}
  \label{retrograde-transport}
\end{figure}

\subsection{Dynein structure}

\begin{figure}[h]
  \centering
  \includegraphics[width=.65\textwidth,keepaspectratio]{../../figures/dynein-artist-rendition.jpg}
  \caption{Artist's rendition of motor-MT complex.}{Artist rendition of the dynein motor bound to a microtubule. One monomer is bound to the MT, and one is raised up. Modified. Source: Lander Lab, The Scripps Research Institute.}
  \label{dynein-artist-rendition-2}
\end{figure}

\textit{eventually find a different figure}

As shown in the crystal structure in Figure XXX, a dynein motor is a fusion of two identical subunits, or monomers, each with binding, stalk, motor and tail subdomains. Each of these subunits has a unique purpose for the motor.

\textbf{Binding domain}
Microtubule binding domains (MTBDs) allow dynein to bind to the microtubule. 

%% \begin{figure}[h]
%%   \centering
%%   \includegraphics[width=.65\textwidth,keepaspectratio]{../../figures/mtbd-complex.png}
%%   \caption{Crystal structure of the MTBD-MT complex. Modified from Redwine \textit{et. al} \cite{redwineMTBDcomplex}.}
%%   \label{dynein-artist-rendition-2}
%% \end{figure}


This could be fixed up...probably remove multiple of these sections

\subsection{How does it behave?}
Dynein's stepping pattern is distinct from its partner, kinesin. Kinesin takes very precise 8-nm steps \cite{kinesin-step-size}. In contrast, dynein takes a large range of steps, as shown in Figure \\

\subsection{Why is dynein interesting?}
Studying how nature creates directed motion is interesting in itself. Another interesting question is how, given the large number of similarities between motors kinesin and dynein, does dynein achieve such a radically different stepping pattern than kinesin? Elucidating how the structural differences between the motors give rise to the differences in dynamics would provide lots of important information on how the machines fine-tune their stepping.\\

The major difference between kinesin and dynein is their size. Kinesin proteins are generally 100-200 kD in mass, whereas dynein is roughly 1mD. As shown in Figure \ref{fig:kinesin-vs-dynein}, the size and volume differences between proteins is very apparent. Dynein is about 20nm from MTBD to tail, whereas kinesin is ----. Another key difference is the separation between active sites on either protein. Kinesin binds to the MT with a foot domain, colored blue in Figure \ref{fig:kinesin-vs-dynein}. It hydrolyzes ATP at the residues colored pink. The separation between these domains is roughly ---nm. In contrast, Dynein binds to the MT with its MTBD, colored blue, and hydrolyses ATP in its motor domain, residues colored pink. This distance is close to 20nm in length - much longer than kinesin. Another difference is the way the motors interact with the microtubule. Both motors bind to the MT using domains. However, for dynein the motor-MTBD-MT angle is ---, whereas for kinesin, it is ---.\\

Despite these differences, the two motors implement very similar chemical cycles to achieve motion. Both feet begin bound to the microtubule, then one unbinds. This foot diffuses forward, then rebinds to the microtubule

%% \begin{figure}[h]
%%   \centering
%%   \includegraphics[width=.65\textwidth,keepaspectratio]{../../figures/kinesin-vs-dynein}
%%   \caption{Crystal structure of the MTBD-MT complex. Modified from Redwine \textit{et. al} \cite{redwineMTBDcomplex}.}
%%   \label{fig:kinesin-vs-dynein}
%% \end{figure}

\section{Dynein Stepping Background}
Biophysicists and molecular biologists have conducted experiments over the past twenty years which shed light on how the dynein motor might walk. The full mechanism is not known, but results of such experiments suggest a certain vague picture of how dynein generates motion.\\

\subsection{ATP cycle influences conformational state}
Dynein is known to occupy multiple discrete conformational states depending on its environment. In particular, the concentration of ATP and microtubules is known to alter the shape dynein takes on. A FRET (fluorescence resonance energy transfer) study attached fluorophores to dynein's motor and tail and measured the average distance between the two \cite{FRETstatepaper}. It was found that average distance increased or decreased depending on the nucleotide bound to dynein, indicating the motor takes on different conformations for different bound nucleotides. Dynein unbound to nucleotide, or frozen in ATP or ADP-bound states, had a similar tail-motor distance. However, when the motor was stuck in an ADP-Pi state, with ATP hydrolyzed but phosphate not yet released, the tail-motor distance was increased. This suggests that dynein starts its cycle with no nucleotide, binds ATP, hydrolyzes ATP, enters the ADP-Pi state and changes conformation to a far tail-motor distance, then releases phosphate, moves back to its original close tail-motor conformation, then ejects ADP and starts over.\\

Studies suggest that near/far tail-motor conformation changes are not due to lengthening of the stem linking the two domains, but rather a change in position of the head relative to the trail \cite{FRETstatepaper} \cite{burgess-paper}. Cryo-electron microscopy images of the dynein motor shown in Figure (\ref{fig:cryo-em-angles}) demonstrate the differences in conformation in the nucleotide-free and ADP-Pi state.\\

\begin{figure}[h]
  \centering
  \includegraphics[width=.65\textwidth,keepaspectratio]{../../figures//burgess-dynein-angles}
  \caption{Cryo-EM microscopy of dynein conformational states.}{Cryo-electron microscopy images of dynein in two different nucleotide states. Left: dynein frozen in ADP-Pi state. Middle: dynein without nucleotide. Right: two states superimposed on one another. Bottom: cartoons representing how dynein may bind to the MT in either state. From \cite{burgess-paper}.}
  \label{fig:cryo-em-angles}
\end{figure}

Crystal structures of cytoplasmic dynein-2, a relative of dynein, indicate that ATP-Pi binding to the motor domain induces a conformation change in the motor domain, which alters the angle at which the motor binds to the tail \cite{carter-paper} (make a Chimaera figure where the pre/poststroke motors are docked showing this). Taken together, these results indicate that as the motor binds and hydrolyzes ATP, it undergoes a conformation change which physically moves the motor relative to the tail.\\

\textbf{TODO: look at \cite{nicastro} and \cite{schmidt-carter} to see if they can add to this discussion}

\subsection{Microtubule and nucleotide states are coupled}
Dynein is known to bind to the microtubule at a location known as the $\alpha$-$\beta$ polymerization site \cite{redwineMTBDcomplex}. This site occurs roughly every 8nm, indicating dynein is restricted to step sizes which are multiples of 8nm.\\

The nucleotide state of the motor is not the only determiner of dynein conformation; microtubule binding state has also been shown to influence conformation. A study on the MTBD-MT complex indicates the stalk connecting the motor and MTBD can take on two major conformations, depending on whether the MTBD is bound to the MT or not \cite{mt-atp-coupling}. The conformation of the stalk has been shown to communicate information between the motor and MTBD. In the presence of microtubules, the ATPase rate of the motor is increased 25-fold. This indicates the stalk communicates MT-binding state of the MTBD to the motor, increasing its ATP hydrolysis rate. The communication happens the other way, as well. Treating dynein with ATP causes MT-binding rate to drop 15-fold, indicating ATP-binding at the motor decreases MT-binding rate at the MTBD.\\

In addition to a bidirectional linking between ATPase rate and MT-binding, stalk state has been shown to directly influence motor conformation \cite{mt-atp-coupling}. When the stalk is allowed to freely transition between bound and unbound states, the motor can also transition between near and far tail/motor states. However, when the stalk is frozen in a particular conformation, the motor loses its transitioning ability. This suggests that the conformations of the two systems are tightly linked.\\

\subsection{Stepping pattern}
Several studies have been done on the dynein motor's stepping pattern using selectively-labeled motor domains \cite{reck2006single} \cite{weihongpaper}. These studies show dynein's stepping pattern to be stochastic and weakly forward-directed. As shown in Figure (\ref{fig:weihong-steps}), dynein takes many backwards steps, and also many steps with components perpendicular to the microtubule. This indicates the process dynein uses to generate motion has a stochastic component to it. It is likely that the physical states of subcomponents in the motor, and/or the effects of Brownian motion due to water particle collision, play a role in dynein's stepping mechanism.\\

\begin{figure}[h!]
  \centering
  \begin{minipage}[b]{0.45\textwidth}
    \includegraphics[width=\textwidth]{../../figures/weihong-step-size}
    \caption{Experimental dynein step size histogram.}{Size of steps taken by dynein motor parallel to microtubule. Positive axis corresponds to steps towards the microtubule minus-end. Figure modified from \cite{weihongpaper}.}
  \end{minipage}
  \hspace{.2cm}
  \begin{minipage}[b]{0.45\textwidth}
    \includegraphics[width=\textwidth]{../../figures/weihong-off-axis-steps}
    \caption{Dynein experimental off-axis stepping size.}{Size of ``off-axis'' steps taken by the dynein motor perpendicular to the microtubule. Figure modified from \cite{weihongpaper}.}
  \end{minipage}
  \label{fig:weihong-steps}
\end{figure}

\section{Dynein models}
Data presented in the previous sections paints a picture of dynein as a collection of interconnected parts, each cycling through its own collection of states and influencing other parts. Somehow the protein itneracts with nucleotides and microtubules in a way that allows it to reliably move through the cell. Models presented below take these facts and string them together into cohesive cycles which might explain how dynein walks. It is important to note that these models are hypotheses.\\

\subsection{Cianfrocco Model}
Cianfrocco \textit{et. al} \cite{cianfroccoreview} propose dynein achieves motion via transitioning through the cycle of states shown in Fig. \ref{mech-cycle}. This model will hereon be referred to as the Cianfrocco model. The model claims the primary events in the dynein step are as follows. First, starting with both binding domains attached to the MT, ATP binds to the motor domain, which causes it to change conformation. ATP hydrolysis to ADP-Pi causes further conformation changes. These changes cause the binding domain to release the microtubule, meaning only one binding domain is MT-bound. The free binding domain then undergoes another conformation change known as the ``priming stroke,'' which changes the orientation of the binding domain with respect to the MT. Finally the binding domain rebinds to the MT, causing a final conformation change in the motor which returns it to the beginning of the cycle. The exact order of these events is not precisely known.\\

\begin{figure}[h!]
  \centering
  \includegraphics[width=.90\textwidth,keepaspectratio]{../../figures/mechanochemical-cycle.jpeg}
  \newcommand\bothbound{\textcolor{blue}{BOTHBOUND}}
  \newcommand\onebound{\textcolor{red}{ONEBOUND}}
  \newcommand\transition{\textcolor{green}{transition?}}
  %% \begin{tabular}{rll}
  %%   \bothbound &
  %%   1 & \begin{minipage}{5in}
  %%     Bound to microtubule and waiting for ATP to arrive.  The
  %%     conformation is ``post-powerstroke''.
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \bothbound &
  %%   2 & \begin{minipage}{5in}
  %%     ATP has bound to the motor at AAA1, but nothing else has changed.
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \transition &
  %%   3 & \begin{minipage}{5in}
  %%     ``The ring has closed'' in the motor, whatever that means.
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \bothbound &
  %%   4 & \begin{minipage}{5in}
  %%     The affinity to the microtubule is dropped, so it is likely to pop off soon.
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \onebound &
  %%   5 & \begin{minipage}{5in} It has popped off of the microtubule!
  %%     Onebound! The equilibrium angle is also changed.  The ATP has
  %%     been hydrolized, but the phosphate is still bound to the motor.
  %%     ``Priming stroke?''
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \onebound &
  %%   6 & \begin{minipage}{5in} It is now bound to the microtubule, but
  %%     with low affinity, and without any loss of phosphate. Question:
  %%     does this happen many times? If so it may still be onebound.
  %%     Otherwise it is probably bothbound.
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \bothbound &
  %%   7 & \begin{minipage}{5in}
  %%     Now it is seriously bound to the microtubule, and the motor
  %%     domain has changed conformation.  This is now ``high affinity.''
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \bothbound &
  %%   8 & \begin{minipage}{5in}
  %%     Phosphate has been released, and our equilibrium angles have
  %%     been changed.  This is the ``power stroke.''
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \bothbound &
  %%   1 & \begin{minipage}{5in}
  %%     The ADP has been released from the motor domain, so it can now
  %%     bind ATP.  Still ``high affinity,'' and still in the
  %%     ``power-stroke'' conformation.
  %%   \end{minipage}\vspace{0.5em}\\
  %%   \bothbound &
  %%   9 & \begin{minipage}{5in}
  %%     While stuck in this ``normal bound'' state, another site on the
  %%     motor AAA3 can lose its ADP, which leaves the motor in a state
  %%     where it cannot directly move on to step 2 (until after
  %%     reversing this).  Controversial?
  %%   \end{minipage}
  %% \end{tabular}
  \caption{Cianfrocco mechanochemical cycle.}{Cycle of states the dynein motor goes through during motion in the Cianfrocco model \cite{cianfroccoreview}.}
  \label{mech-cycle}
\end{figure}

%% \subsection{Other models}
%% Find other models and put them here

%% \section{Brownian dynamics}
%% New section which describes how BD works.

\section{This work}
\subsection{Necessary and sufficient criteria for motion}
One of the principle goals of researchers studying the dynein motor is establishing how it walks. This can be a difficult task, since direct observation of the motor walking is not currently experimentally possible. The next best thing, then, is building a model which both only does all the things dynein does, and generates motion. Finding such a system would be very useful for forming hypotheses on how dynein walks. Another requirement of the system would be that it be as simple as possible while still explaining all the data. By Occam's Razor, such systems are the most likely to represent real dynein. Thus, finding the necessary and sufficient criteria for dynein's motion is a big goal in the field.\\

A good starting place for finding the minimum set of motile criteria would be preexisting models, like the Cianfrocco model. This model describes a set of states which logically lead to each other and are supported by experiment. It seems intuitive that a motor which obeyed the Cianfrocco cycle would walk. There are multiple ways to verify this. A completely synthetic biological motor could be designed which obeys the cycle, and its motile properties could be studied. This seems incredibly difficult. Much easier would be to create a physical model of such a cycle, simulate it, and see if it obeys similarly to experimentally observed dynein. If the physical model both behaves according to the laws of physics and captures experimental dynein behavior well, this would be good evidence that such a model describes how dynein actually walks.\\

\subsection{Goals}
The goal of this work is to construct a minimum physical model of dynein protein based on the theoretical Cianfrocco model. The model should behave as physics predicts a protein to behave; namely, it should obey Brownian dynamics and conservation of energy. It should also roughly capture the state transitions and conformation changes of the Cianfrocco model. Also the model should be as simple as possible, to assure that each part of the model is necessary. If too many extra features are added, the model will not be minimal and not achieve its purpose.\\

This model will then be converted into a computer program, tested for compliance with the laws of physics, and simulated. Information on the long-term behavior of such a simulation will then be used to evaluate how well the model describe's experimental dynein's behavior.\\


%% \textbf{Things which are known about dynein which support the Cianfrocco model}
%% \begin{itemize}
%% \item AAA1 only ATPase necessary for motility, which is why we don't include the other ATPases in our model \cite{cianfroccoreview}
%% \item AAA5 mediates stalk register changes Schmidt H, Zalyte R, Urnavicius L, Carter AP. 2014. Structure of human cytoplasmic dynein-2 primed for
%%   its power stroke. Nature 518:435–38
%% \item Linker moves as response to AAA1 nucleotide binding Burgess SA, Walker ML, Sakakibara H, Knight PJ, Oiwa K. 2003. Dynein structure and power stroke. Nature
%%   421:715–8
%% \item Crystal structures with Vanadium-Adp show dynein in the pre-powerstroke state, hinting that ADP-Pi-bound dynein takes on a different conformation than ADP-Pi free dynein
%% \end{itemize}

%% \textit{It is interesting to note that the hydrolysis of ATP is not actually used to directly power the motor -- binding AMP-PMP, a nonhydrolyzable analog of ATP, can also cause the motor to change from state I to state II \cite{FRETstatepaper}. The specialness of ATP is that it can be broken down into ADP, which causes another conformation change back to state I. Thus, something something the transfer of energy from nucleotide to motor is in its shape, not in its chemistry.}\\

\chapter{Theory}
\label{chap:Theory}

\section{Model features}
The purpose of this project is to find if a minimal set of features is capable of reproducing dynein's stepping behavior. To test this hypothesis, these features are put into a model and simulated. A schematic of the model is pictured in Figure \ref{fig:model-intro} and described below.\\

\textit{Add a figure (maybe here?) of all the events which are known to happen in dynein, along with our mechanochemical cycle. It would basically show all events occuring at two times: unbinding or rebinding.}\\

%% \begin{itemize}
%% \item Dynein is a collection of spherical domains joined together by massless rods
%% \item These rods are rigid
%% \item Each joint in the motor feels a spring force about some equilibrium joint angle
%% \item Dynein can be in two states: onebound and bothbound
%% \item Onebound has one domain bound to the floor and takes on onebound equilibrium angles
%% \item Bothbound has two domains bound to the floor and takes on bothbound equilibrium angles
%% \item Transitions between the two states are based solely on spring energies between the two models
%% \item Brownian dynamics describes the behavior of the domains in time
%% \item Binding at $\alpha-\beta$ tubulin binding sites is not necessary for stepping
%% \item Stepping can be described by a 2D model; forces in the MT-perpendicular direction are not necessary for stepping
%% \end{itemize}

%% \begin{figure}[h]
%%   \centering
%%   \includegraphics[width=.65\textwidth,keepaspectratio]{../../figures/ob_fig}
%%   \caption{Onebound dynein model.}{}
%%   \label{fig:model-intro}
%% \end{figure}

\subsection{Spherical domains connected by massless rods}
A key feature of the model is that each domain of the protein (two feet, twp motors and a tail) is treated as a sphere. Each sphere is connected to one or two other spheres, as shown in Figure (\ref{fig:model-intro}). This simplification is made to make the math easier and allow a simple definition of dynein's conformation.\\

One quickly notices by glancing at Figure (\ref{fig:crystal-structure-model-imposed}), a visualization of actual dynein, that modelling the tail and feet domains as spheres is a stretch. The main point of the simplification is not to capture the shape of dynein, but the conformation. It is undefined to discuss the distance between the two motor domains in real dynein, because the motors are made up of hundreds of amino acids, and the average inter-domain distance between amino acids is comparable to the intra-domain distance. However it is very easy to describe the difference between two spheres.\\

Each sphere has an associated drag factor $\gamma$, which describes the resistance it feels to motion. This drag factor is calculated based on the radius $R$ of the spheres, and is given by Stoke's Law:
%
\begin{equation}
  \gamma = 6\pi\nu R
\end{equation}
%
Thus R becomes a parameter input to the model.\\

\subsection{Rigid rods}
The model assumes connected domains are rigidly attached to one another by rods. This assumption makes the math easier, and is not necessarily based on biology. One study found that the standard deviation of stalk and stem lengths in \textit{in vitro} dynein was roughly $1nm$ \ref{burgess-paper}. However a contrasting piece of evidence showed the coiled coil, a common protein structure which links the MTBD and motor domains in dynein, can stretch in myosin, another motor protein, by 2.5x \ref{myosin-cc-elasticity}.

In reality, all molecules have a degree of elasticity. A dynein model by Sarlah \textit{et. al.} models the dynein stalk as an elastic rod \cite{sarlahmodel}. However, the timescale of harmonic stalk oscillations is likely much quicker than that of the mechanochemical cycle, so it is reasonable to model the stalk as a rigid rod at the stalk's average extension length.\\

\subsection{Angular springs}
Each domain has a spring constant, $c_b, c_m$ or $c_t$ for binding, motor or tail, which decides the energy associated with that domain angle. For example, the tail domain energy is given by
%
\begin{equation}
  \Delta G_t = \frac12c_t\left(\theta_t - \theta_t^{eq}\right)^2
\end{equation}

\subsection{Two-state model}
The model assumes MT-bound dynein can assume one of two states: one where both MTBDs are bound, and one where only one MTBD is bound to the MT. These states are also differentiated by different motor equilibrium angles, $\Theta_{m}^{bb}$ and $\Theta_{m}^{ob}$, and different transition rates. Biological dynein likely passes through a series of permutations of nucleotide, MT affinity and conformational states as it travels through its mechanochemical cycle. This model makes the assumption that the mechanochemical cycle can be broken down into a cycle of prestroking to move the unbound foot forward, brownian walking to place the foot on the microtubule, then powerstroking to move the other domains forward.\\

\begin{figure}[h]
  \centering
  \includegraphics[width=.45\textwidth]{../../figures/code-onebound}
  \includegraphics[width=.45\textwidth]{../../figures/code-bothbound}
  \caption{Dynein model schematics.}{Representations of onebound (left) and bothbound (right) dynein models.}
  \label{ob_fig}
\end{figure}

\textbf{todo: Add springs to joints!}

\subsection{Transitioning between states}
The model transitions between states with a probability dependent on both the energy difference between the two states and the physical separation between the MT and binding MTBD. The energy difference between states A and B is given by:
%
\begin{equation}
  \Delta G_{A\rightarrow B} = \sum_{i=0...N_B}^{B} \frac{1}{2}c_i\left(\theta_i-\Theta_i^B\right)^2 - \sum_{j=0...N_A}^{A} \frac{1}{2}c_j\left(\theta_j-\Theta_j^A\right)^2
\end{equation}
%
Transition energies are used to calculate transition rates, as given by the Arhennius equation $k=Ae^{-\beta\Delta G}$, where $\beta = (k_BT)^{-1}$. In this equation $\Delta G$ is the total energy of transition between states and A is a constant known as the preexponential factor with units of $s^{-1}$.\\
%
\begin{equation}
  k_b = Ae^{\frac{-\Delta G}{k_BT}} = Ae^{\frac{-\Delta G_{conf}}{k_BT}}e^{\frac{-\Delta G_{bind}}{k_BT}}
\end{equation}
%
where $\Delta G_{conf}$ is the change in spring energy in going from bothbound to onebound and $\Delta G_{bind}$ is the energy gained from binding to the microtubule. Because binding energy for the MT-MTBD complex is hard to find, the following substitution is made:
%
\begin{equation}
  k_b = Ae^{\frac{-\Delta G}{k_BT}} = Be^{\frac{-\Delta G_{conf}}{k_BT}}
\end{equation}
%
Thus $B$ becomes a parameter of the model.\\

\subsection{Brownian dynamics}
The Brownian dynamics equation describes the motion of high-drag objects which spend most of their time at terminal velocity, and is given by:
%
\begin{equation}
  \dot{x} = \frac{1}{\gamma}\left(F_{net} + R\right)
\end{equation}
%
where $\gamma$ is the drag constant of the object and $R$ is a random Brownian force which describes the effects of water particle collision. $R$ is a Gaussian-sampled zero-centered force with variance given by:
%
\begin{equation}
  R_{\sigma^2} = \sqrt{\frac{2*kb*T*\gamma}{dt}}
\end{equation}
%
Dynein is in the range of objects which BD describes well. (citation needed)

\subsection{Binding to microtubule}
The model ``steps'' whenever an MTBD diffuses near the microtubule. \textit{In vivo}, dynein binds to a site on the $\alpha-\beta$ tubulin polymer microtubule which repeats roughly every 8nm. Thus, all dynein steps should be multiples of $8$nm. This model ignores this requirement and allows stepping to occur at any length. The assumption is that \textit{in vivo} dynein takes 8nm steps because of its size and stepping angle. Thus the 8nm step length should be reproduced by the model without explicit coding.\\

\subsection{2D model}
The coordinate system for this model is two-dimensional. Thus interactions between the two motors and MTBDs are ignored. This assumption is made to simplify the model. Other dynein models, like the winch model \ref{sarlahmodel}, add in attractive and repulsive forces between the two motors which act in three dimensions. It is not clear why such forces are necessary for dynein motion, and so they are omitted.\\

Side stepping of dynein is ignored in this model. Stepping perpendicular to the MT occurs in dynein \cite{weihongpaper}, as shown in Figure \ref{fig:weihong-side-stepping}. This model assumes it is a symptom, and not an influencer of, of dynein's forward-backward stepping mechanism.

\begin{figure}[h]
  \centering
  \includegraphics[width=.45\textwidth]{../../figures/weihong-side-stepping}
  \caption{Dynein experimental off-axis stepping histogram.}{Angular histogram of off-axis stepping of dynein, as found by \cite{weihongpaper}. 0 degrees corresponds to a step parallel to the MT, and 270 degrees a step purely to the right. Modified.}
  \label{fig:weihong-side-stepping}
\end{figure}

\section{Mathematical models}
To simulate a protein's passage through the above mechanochemical cycle, two mathematical models are needed: one for the prestroke, MTBD-detached state, and one for the poststroke, MTBD-bound state. The purpose of each model is to calculate the position and velocity of each domain at an arbitrary time.\\

\subsection{Prestroke onebound model}
Prestroke dynein is referred to as onebound, since one motor domain is bound to the MT in this state, and the other unbound. A spatial representation of onebound dynein is shown in Figure \ref{ob_fig}. 

%% \begin{figure}[h]
%%   \centering
%%   \includegraphics[width=.45\textwidth]{../../figures/ob_fig.pdf}
%%   \caption{Spatial representation of onebound model.}
%%   \label{ob_fig}
%% \end{figure}

TODO: make onebound model figure, label domain names (MTBD motor tail), Ls, Lt, microtubule\\

Each domain is represented by a point, and is connected to other domains via fixed-length rods. Angles represent the degrees of freedom of the model. Since the protein is a dimer, there are two copies of the MTB, motor and tail domains. The tails are fused together and so represented as a single point. The motor and MTB domains are given a ``bound'' and ``unbound'' designation to differentiate them. The ``bound'' domains are the MTBD attached to the MT and its corresponding motor domain. The unbound domains are the unbound MTBD and its corresponding motor domain.\\

\paragraph{Angular and Cartesian coordinates}

The four angles $\Theta_{bb}, \Theta_{bm}, \Theta_{um}$ and $\Theta_{ub}$, corresponding to bound binding, bound motor, unbound motor and unbound binding angle, together describe any possible conformation the system can take on. The domain of $\Theta_{bb}$ has a restricted domain of $[0,\pi]$ to prevent below-MT conformations, but the other angles have domains of $[0,2\pi]$. Each angle is defined relative to the horizontal axis. The position of the bound binding domain $X_{bb}$ and the four domain angles are all that is needed to determine the position of the protein at any time.\\

%% \begin{figure}[h]
%%   \centering
%%   \begin{tabular}{c}
%%     \begin{lstlisting}[language=C++]
%%       class Dynein_onebound{
%%         double &$\Theta_{bb}, \Theta_{bb}, \Theta_{bb}, \Theta_{bb}$&
%%         double &$\dot{\Theta}_{bb}, \dot{\Theta}_{bb}, \dot{\Theta}_{bb}, \dot{\Theta}_{bb}$&
%%         double &$X_{bb}$&
%%       }
%%     \end{lstlisting}
%%   \end{tabular}
%%   \caption{Pseudocode of the onebound Dynein model.}
%%   \label{ob_struct}
%% \end{figure}

\paragraph{$\dot{\Theta}_{ub}, \dot{\Theta}_{um}, \dot{\Theta}_{bm}, \dot{\Theta}_{bb}$ calculation}

The following equations allow recursive calculation of the positions of each domain in cartesian coordinates:

\noindent\begin{minipage}{0.49\linewidth}
\begin{align}
  X_{bm} &= X_{bb}+L_{s}\cos(\Theta_{bb}) \\
  X_{t}  &= X_{bm}+L_{t}\cos(\Theta_{bm}) \\
  X_{fm} &= X_{t} - L_{t}\cos(\Theta_{fm}) \\
  X_{fb} &= X_{fm} - L_{s}\cos(\Theta_{fb})
\end{align}
\end{minipage}
\begin{minipage}{0.49\linewidth}
\begin{align}
  Y_{bb} &= 0 \\
  Y_{bm} &= Y_{bb}+L_{s}\sin(\Theta_{bb}) \\
  Y_{t}  &= Y_{bm}+L_{t}\sin(\Theta_{bm}) \\
  Y_{fm} &= Y_{t} - L_{t}\sin(\Theta_{fm}) \\
  Y_{fb} &= Y_{fm} - L_{s}\sin(\Theta_{fb})
\end{align}
\end{minipage}
\vspace{.5cm}

$L_s$ and $L_t$ correspond to lengths of each interdomain rod, and the $t$ subscripts refer to tail domain coordinates.\\

\paragraph{Angular and Cartesian velocities}
The goal is to express angular velocities $\dot{\Theta}_{bb}, \dot{\Theta}_{bb}, \dot{\Theta}_{bb}$ and $\dot{\Theta}_{bb}$ in terms of known quantities. To begin, the cartesian velocities of each domain can be calculated from the above position equations in a similar recursive manner. A subset of the equations are shown here:

\noindent\begin{minipage}{0.49\linewidth}
\begin{align}
  \dot{X}_{bb} &= 0 \\
  \dot{X}_{bm} &= \dot{X}_{bb} - L_{s}\sin(\Theta_{bb})\dot{\Theta}_{bb} \label{cartesian-bmx}\\
  \dot{X}_{t } &= \dot{X}_{bm} - L_{t}\sin(\Theta_{bm})\dot{\Theta}_{bm}
  %% \dot{X}_{fm} &= \dot{X}_{t } + L_{t}\sin(\Theta_{fm})\dot{\Theta}_{fm} \\
  %% \dot{X}_{fb} &= \dot{X}_{fm} + L_{s}\sin(\Theta_{fb})\dot{\Theta}_{fb}
\end{align}
\end{minipage}
\begin{minipage}{0.49\linewidth}
\begin{align}                                                                          
  \dot{Y}_{bb} &= 0 \\                                                        
  \dot{Y}_{bm} &= \dot{Y}_{bb} + L_{s}\cos(\Theta_{bb})\dot{\Theta}_{bb} \\
  \dot{Y}_{t}  &= \dot{Y}_{bm} + L_{t}\cos(\Theta_{bm})\dot{\Theta}_{bm}
  %% \dot{Y}_{fm} &= \dot{Y}_{t } - L_{t}\cos(\Theta_{fm})\dot{\Theta}_{fm} \\
  %% \dot{Y}_{fb} &= \dot{Y}_{fm} - L_{s}\cos(\Theta_{fb})\dot{\Theta}_{fb}
\end{align}
\end{minipage}
\vspace{.5cm}

Another way to express these cartesian velocities is using the Brownian dynamics equation $\dot{X} = \frac1\gamma F_{net} + R$:

\begin{align}  
  \dot{X}_{bm} &= \frac{1}{\gamma_m} \Big(F_{xml} + - \lambda_{bs}(X_{bm} - X_{bb}) + \lambda_{bt}(X_{t } - X_{bm}) \Big) + R_{xml} \label{brownian-bmx}\\
  \dot{X}_{t } &= \frac{1}{\gamma_t} \Big(F_{xt } + - \lambda_{bt}(X_{t } - X_{bm}) + \lambda_{ft}(X_{fm} - X_{t }) \Big) + R_{xt }
  %% \dot{X}_{fm} &= \frac{1}{\gamma_m} \Big(F_{xmr} + - \lambda_{ft}(X_{fm} - X_{t }) + \lambda_{fs}(X_{fb} - X_{fm}) \Big) + R_{xmr} \\
  %% \dot{X}_{fb} &= \frac{1}{\gamma_b} \Big(F_{xbr} + - \lambda_{fs}(X_{fb} - X_{fm}) \Big) + R_{xbr}
\end{align}

%% \begin{align}  
%%   \dot{Y}_{bm} &= \frac{1}{\gamma_m} \Big(F_{yml} + - \lambda_{bs}(Y_{bm} - Y_{bb}) + \lambda_{bt}(Y_{t } - Y_{bm}) \Big) + R_{yml} \\
%%   \dot{Y}_{t}  &= \frac{1}{\gamma_t} \Big(F_{yt } + - \lambda_{bt}(Y_{t } - Y_{bm}) + \lambda_{ft}(Y_{fm} - Y_{t }) \Big) + R_{yt } \\
%%   \dot{Y}_{fm} &= \frac{1}{\gamma_m} \Big(F_{ymr} + - \lambda_{ft}(Y_{fm} - Y_{t }) + \lambda_{fs}(Y_{fb} - Y_{fm}) \Big) + R_{ymr} \\
%%   \dot{Y}_{fb} &= \frac{1}{\gamma_b} \Big(F_{ybr} + - \lambda_{fs}(Y_{fb} - Y_{fm}) \Big) + R_{ybr}
%% \end{align}

where $\gamma_n$ is a drag coefficient for the binding, motor or tail domain with units of mass per second. $F_{xn}$ is the x component of external force on each domain $n$ due to various factors. $\lambda_{12}\left(X_1 - X_2\right)$ is the x component of internal force on domain $2$ due to domain $1$, where $\lambda$ is a tension coefficient with units of mass per second per second. $R_{xn}$ is the Brownian coefficient representing motion due to solvent collision, with units of velocity.\\

These two sets of equations, cartesian and Brownian, can be equated to get more interesting equations. For example, Eq (\ref{cartesian-bmx}) and Eq (\ref{brownian-bmx}) can be equated. This equating, combined with expanding the recursive velocity definitions leads to new equations, some of which are shown here:

\begin{align}
  &-L_s\sin(\theta_{bb})\dot{\theta}_{bb} = \frac{1}{\gamma_m}F_{xml} + -\frac{1}{\gamma_m}\lambda_{bs}(X_{bm} - X_{bb}) + \frac{1}{\gamma_m}\lambda_{bt}(X_{t } - X_{bm}) + R_{bmx} \label{ob_system_first}\\
  &-L_s\sin(\theta_{bb})\dot{\theta}_{bb} - L_t\sin(\theta_{bm})\dot{\theta}_{bm} = \frac{1}{\gamma_t}F_{xt } + -\frac{1}{\gamma_t}\lambda_{bt}(X_{t } - X_{bm}) + \frac{1}{\gamma_t}\lambda_{ft}(X_{um} - X_{t }) + R_{tx}
  %% &-L_s\sin(\theta_{bb})\dot{\theta}_{bb} - L_t\sin(\theta_{bm})\dot{\theta}_{bm} + L_t\sin(\theta_{um})\dot{\theta}_{um} = \frac{1}{\gamma_m}F_{xmr} + -\frac{1}{\gamma_m}\lambda_{ft}(X_{um} - X_{t }) + \frac{1}{\gamma_m}\lambda_{fs}(X_{ub} - X_{um}) + R_{bmx} \\
  %% &-L_s\sin(\theta_{bb})\dot{\theta}_{bb} - L_t\sin(\theta_{bm})\dot{\theta}_{bm} + L_t\sin(\theta_{um})\dot{\theta}_{um} + L_s\sin(\theta_{ub})\dot{\theta}_{ub} = \frac{1}{\gamma_b}F_{xbr} + -\frac{1}{\gamma_b}\lambda_{fs}(X_{ub} - X_{um}) + R_{bbx}
\end{align}

%% \begin{align}
%%   &L_s\cos(\theta_{bb})\dot{\theta}_{bm} = \frac{1}{\gamma_m}F_{yml} + -\frac{1}{\gamma_m}\lambda_{bs}(Y_{bm} - Y_{bb}) + \frac{1}{\gamma_m}\lambda_{bt}(Y_{t } - Y_{bm}) + R_{umy} \\
%%   &L_s\cos(\theta_{bb})\dot{\theta}_{bb} + L_t\cos(\theta_{bm})\dot{\theta}_{bm} = \frac{1}{\gamma_t}F_{yt } + -\frac{1}{\gamma_t}\lambda_{bt}(Y_{t } - Y_{bm}) + \frac{1}{\gamma_t}\lambda_{ft}(Y_{um} - Y_{t }) + R_{ty} \\
%%   &L_s\cos(\theta_{bb})\dot{\theta}_{bb} + L_t\cos(\theta_{bm})\dot{\theta}_{bm} - L_t\cos(\theta_{um})\dot{\theta}_{um} = \frac{1}{\gamma_m}F_{ymr} + -\frac{1}{\gamma_m}\lambda_{ft}(Y_{um} - Y_{t }) + \frac{1}{\gamma_m}\lambda_{fs}(Y_{ub} - Y_{um}) + R_{bmy} \\
%%   &L_s\cos(\theta_{bb})\dot{\theta}_{bb} + L_t\cos(\theta_{bm})\dot{\theta}_{bm} - L_t\cos(\theta_{um})\dot{\theta}_{um} - L_s\cos(\theta_{ub})\dot{\theta}_{ub} = \frac{1}{\gamma_b}F_{ybr} + -\frac{1}{\gamma_b}\lambda_{fs}(Y_{ub} - Y_{um}) + R_{bby} \label{ob_system_last}
%% \end{align}

A total of eight coupled differential equations are formed from this procedure. These eight equations form a system of equations with eight unknowns: $\dot{\Theta}_{bb}, \dot{\Theta}_{bb}, \dot{\Theta}_{bb}$ and $\dot{\Theta}_{bb}$, and the four tension coefficients $\lambda_{bs}, \lambda_{bt}, \lambda_{um}$ and $\lambda_{ub}$. This system is more compactly represented as:
{\small
  \[
  \hspace{-1.0cm}
  \resizebox{0.85\linewidth}{!}{%
  $\begin{pmatrix}
    L_s\sin\theta_{bb} & 0 & 0 & 0 & -\gamma_m (X_{bm} - X_{bb}) & \gamma_m (X_{t } - X_{bm}) & 0 & 0\\
    L_s\sin(\theta_{bb}) & L_t\sin(\theta_{bm}) & 0 & 0 & 0 & -\gamma_t (X_{t } - X_{bm}) & \gamma_t (X_{um} - X_{t }) & 0\\
    L_s\sin(\theta_{bb}) & L_t\sin(\theta_{bm}) & -L_t\sin(\theta_{um}) & 0 & 0 & 0 & -\gamma_m (X_{um} - X_{t }) & \gamma_m (X_{ub} - X_{um})\\
    L_s\sin(\theta_{bb}) & L_t\sin(\theta_{bm}) & -L_t\sin(\theta_{um}) & -L_s\sin(\theta_{ub}) & 0 & 0 & 0 & -\gamma_b (X_{ub} - X_{um})\\
    -L_s\cos(\theta_{bb}) & 0 & 0 & 0 & -\gamma_m (Y_{bm} - Y_{bb}) & \gamma_m (Y_{t } - Y_{bm}) & 0 & 0\\
    -L_s\cos(\theta_{bb}) & -L_t\cos(\theta_{bm}) & 0 & 0 & 0 & -\gamma_t (Y_{t } - Y_{bm}) & \gamma_t (Y_{um} - Y_{t }) & 0\\
    -L_s\cos(\theta_{bb}) & -L_t\cos(\theta_{bm}) & L_t\cos(\theta_{um}) & 0 & 0 & 0 & -\gamma_m (Y_{um} - Y_{t }) & \gamma_m (Y_{ub} - Y_{um})\\
    -L_s\cos(\theta_{bb}) & -L_t\cos(\theta_{bm}) & L_t\cos(\theta_{um}) & L_s\cos(\theta_{ub}) & 0 & 0 & 0 & -\gamma_b (Y_{ub} - Y_{um})\\
  \end{pmatrix}$%
  }
  \begin{pmatrix}
    \dot{\theta}_{bb}\\
    \dot{\theta}_{bm}\\
    \dot{\theta}_{um}\\
    \dot{\theta}_{ub}\\
    \lambda_{bs}\\
    \lambda_{bt}\\
    \lambda_{ut}\\
    \lambda_{us}\\
  \end{pmatrix}
  =
  \begin{pmatrix}
    -F_{bmx} + \gamma_m R_{bmx}\\
    -F_{tx} + \gamma_t R_{tx}\\
    -F_{umx} + \gamma_m R_{umx}\\
    -F_{ubx} + \gamma_b R_{ubx}\\
    -F_{bmy} + \gamma_m R_{bmy}\\
    -F_{ty} + \gamma_t R_{ty}\\
    -F_{umy} + \gamma_m R_{umy}\\
    -F_{uby} + \gamma_b R_{uby}\\
  \end{pmatrix}
  \]
}

This matrix is then solved using the Mathematica computer algebra system, resulting in a set of motion equations which describe the model's trajectory over time, eg $\dot{\Theta_{bm}}(\Theta_{bb}, \Theta_{bm}, \Theta_{um}, \Theta_{ub})$. These motion equations are further described in (\ref{sec:ob-motion-equations}).\\

\subsection{Poststroke bothbound model}
The main difference between the prestroke and poststroke mathematical models is that the latter has a further constraint: both its MTB domains are attached to the MT. This has the effect of diminishing the degrees of freedom of the system from four to two, assuming the motor remains above the MT. A spatial representation of the bothbound model is shown in Figure (\ref{bb_fig}).\\

\begin{figure}[h]
  \centering
  \includegraphics[width=.45\textwidth]{../../figures/code-bothbound.pdf}
  \caption{Model bothbound equilibrium angle defintion.}{}
  \label{bb_fig}
\end{figure}

The bothbound model is very similar in assumptions to the onebound: domains are represented as points connected by rigid rods, and the tails are still fused into a single point. The main differences are the naming scheme and spatial constraints. Dimers can no longer be differentiated by being un/bound, so are now differentiated by their z-position: being near or far from the ``camera.'' This is similar to labeling them as the right or left dimer. Thus there are the near binding (nb), near motor (nm), tail (t), far motor (fm) and far binding (fb) domains. Both the near and far binding domains have their x-coordinate fixed at zero. The system is described completely by two angles: $\Theta_{nm}$ and $\Theta_{fm}$. Angles $\Theta_t, \Theta_{nb}$ and $\Theta_{fb}$ are dependent on these two motor angles.\\

The second MTBD constraint adds another required piece of information the model must track: the x position of the second MTBD. This position is described by $L$, the distance between the near and far binding domains. The pseudocode used to describe the bothbound model are shown in Figure (\ref{bb_struct}).\\

\begin{figure}[h]
  \centering
  \begin{tabular}{c}
    \begin{lstlisting}[language=C++]
      class Dynein_bothbound{
        double &$\Theta_{nm}, \Theta_{fm}$&
        double &$\dot{\Theta}_{nm}, \dot{\Theta}_{fm}$&
        double &$X_{nb, L}$&
      }
    \end{lstlisting}
  \end{tabular}
  \caption{Pseudocode of the bothbound Dynein model.}
  \label{ob_struct}
\end{figure}

\paragraph{$\dot{\Theta}_{nm}, \dot{\Theta}_{fm}$ calculation}

The first step in calculating angular velocities is to define new variables $L_n$ and $L_f$ using the Law of Cosines:

\begin{align}
  L_n &= \sqrt{L_s^2 + L_t^2 - 2L_sL_t\cos{\theta_{nm}}} \\
  L_f &= \sqrt{L_s^2 + L_t^2 - 2L_sL_t\cos{\theta_{fm}}}
\end{align}

These new artificial stalks are not physically relevant, but allow the definition of new angles $\Theta_n, \Theta_{ns}, \Theta_{f}$ and $\Theta_{fs}$, which are useful for calculating the position of the motor domain. The treatment of the near and far versions of these angles is very similar, so only the near angles will be dealt with here. The angles themselves are never dealt with, only their $\sin$s and $\cos$s:

\begin{align}
  \cos\theta_n &= \frac{L^2 + L_n^2 - L_f^2}{2L L_n} \\
  \sin\theta_{n} &= \sqrt{1 - \cos^2\theta_{n}} \\
  \cos\theta_{ns} &= \frac{L_s^2 + L_n^2 - L_t^2}{2L_s L_n} \\
  \sin\theta_{ns} &=
  \begin{cases}
    +\sqrt{1 - \cos^2\theta_{ns}} & \theta_{nm} < \pi \\
    -\sqrt{1 - \cos^2\theta_{ns}} & \theta_{nm} > \pi
  \end{cases}
\end{align}

These values are arrived at through the Law of Cosines and a trigonometric identity. The sign of $\sin\left(\Theta_n\right)$ is restricted to positive values due to the angle's $[0,\pi]$ domain. $\Theta_{ns}$'s domain is $[-\pi,\pi]$, meaning its sign is not restricted. The partial derivatives of these values are used in future calculations, and some are shown here:

\begin{align}
  \frac{\partial \cos\theta_n}{\partial L_n} &= \frac{1}{L} - \frac{L^2 + L_n^2 - L_f^2}{2L L_n^2}\\
  \frac{\partial \cos\theta_n}{\partial L_f} &= -\frac{L_f}{LL_n}\\
  \frac{\partial \sin\theta_n}{\partial L_n} &= \frac{-\cos\theta_n}{\sqrt{1-\cos^2\theta_n}}
  \left(\frac{1}{L} - \frac{L^2 + L_n^2 - L_f^2}{2L L_n^2}\right)
  %% \frac{\partial \sin\theta_n}{\partial L_f} &= \frac{-\cos\theta_n}{\sqrt{1-\cos^2\theta_n}}
  %% \left(\frac{-L_f}{LLn}\right)\\
  %% \frac{\partial \cos\theta_{ns}}{\partial L_n} &= \frac{1}{L_s}
  %% - \frac{L_s^2 + L_n^2 - L_t^2}{2L_sL_n^2}\\
  %% \frac{\partial \cos\theta_{ns}}{\partial L_f} &= 0\\
  %% \frac{\partial \sin\theta_{ns}}{\partial L_n} &=
  %% \begin{cases}
  %%   \frac{-\cos\theta_{ns}}{\sqrt{1-\cos^2\theta_{ns}}}
  %%   \left(\frac{1}{L_s} - \frac{L_s^2+L_n^2-L_t^2}{2LL_n^2}\right) & \theta_{nm} < \pi \\
  %%   \frac{\cos\theta_{ns}}{\sqrt{1-\cos^2\theta_{ns}}}
  %%   \left(\frac{1}{L_s} - \frac{L_s^2+L_n^2-L_t^2}{2LL_n^2}\right) & \theta_{nm} > \pi
  %% \end{cases}\\
  %% \frac{\partial \sin\theta_{ns}}{\partial L_f} &= 0
\end{align}

The position of the motor and tail domains is calculated as follws, using the angle addition trigonometric identity:

\begin{align}
  X_{nm} &= L_s\left(
  \cos\theta_n\cos\theta_{ns} - \sin\theta_n\sin\theta_{ns}
  \right)
  \\
  Y_{nm} &= L_s\left(
  \cos\theta_n\sin\theta_{ns} + \sin\theta_n\cos\theta_{ns}
  \right)
  \\
  X_{t} &= L_n\cos\theta_n\\
  Y_{t} &= L_n\sin\theta_n
\end{align}

The partial derivatives of these values are used later. A selection are shown here:

\begin{align}
  \frac{dX_{nm}}{dL_n} &= L_s\Big(\cos\theta_n\frac{d\cos\theta_{ns}}{dL_n} + \cos\theta_{ns}\frac{d\cos\theta_{n}}{dL_n} - \sin\theta_n\frac{d\sin\theta_{ns}}{dL_n} - \sin\theta_{ns}\frac{d\sin\theta_{n}}{dL_n}\Big)\\
  \frac{dY_{nm}}{dL_n} &= L_s\Big(\cos\theta_n\frac{d\sin\theta_{ns}}{dL_n} + \sin\theta_{ns}\frac{d\cos\theta_{n}}{dL_n} + \sin\theta_n\frac{d\cos\theta_{ns}}{dL_n} + \cos\theta_{ns}\frac{d\sin\theta_{n}}{dL_n} \Big)\\
  \frac{dX_{nm}}{dL_f} &= L_s\Big(\cos\theta_n\frac{d\cos\theta_{ns}}{dL_f} + \cos\theta_{ns}\frac{d\cos\theta_{n}}{dL_f} - \sin\theta_n\frac{d\sin\theta_{ns}}{dL_f} - \sin\theta_{ns}\frac{d\sin\theta_{n}}{dL_f} \Big)
  %% \frac{dY_{nm}}{dL_f} &= L_s\Big(\cos\theta_n\frac{d\sin\theta_{ns}}{dL_f} + \sin\theta_{ns}\frac{d\cos\theta_{n}}{dL_f} + \sin\theta_n\frac{d\cos\theta_{ns}}{dL_f} + \cos\theta_{ns}\frac{d\sin\theta_{n}}{dL_f}\Big)\\
  %% \frac{dX_{t}}{dL_n} &= \cos\theta_n + L_n\frac{\partial \cos\theta_n}{\partial L_n} = \frac{L_n}{L}\\
  %% \frac{dY_{t}}{dL_n} &= \sin\theta_n + L_n\frac{\partial \sin\theta_n}{\partial L_n}\\
  %% \frac{dX_{t}}{dL_f} &= L_n\frac{\partial \cos\theta_n}{\partial L_f} = -\frac{L_f}{L}\\
  %% \frac{dY_{t}}{dL_f} &= L_n\frac{\partial \sin\theta_n}{\partial L_f}\\
\end{align}

These partial derivatives are then used in a system of equations very similar to that in Equations (\ref{ob_system_first}-\ref{ob_system_last}), combining Brownian and coordinate system definitions of domain velocities:

\begin{align}
  \dot{X}_{nm} &= \frac{1}{\gamma} \Big(F_{xml} - \lambda_{ns}(X_{bm} - X_{bb})
  + \lambda_{nt}(X_{t } - X_{bm}) \Big) + R_{xml}
  &=& \frac{\partial X_{nm}}{\partial L_n}\dot{L}_n + \frac{\partial X_{nm}}{\partial L_f}\dot{L}_f\\
  \dot{X}_{t } &= \frac{1}{\gamma} \Big(F_{xt } - \lambda_{nt}(X_{t } - X_{bm})
  + \lambda_{ft}(X_{fm} - X_{t }) \Big) + R_{xt }
  &=& \frac{\partial X_{t}}{\partial L_n}\dot{L}_n + \frac{\partial X_{t}}{\partial L_f}\dot{L}_f\\
  \dot{X}_{fm} &= \frac{1}{\gamma} \Big(F_{xmr} - \lambda_{ft}(X_{fm} - X_{t })
  + \lambda_{fs}(X_{fb} - X_{fm}) \Big) + R_{xmr}
  &=& \frac{\partial X_{fm}}{\partial L_n}\dot{L}_n + \frac{\partial X_{fm}}{\partial L_f}\dot{L}_f
\end{align}

\begin{align}
  \dot{Y}_{nm} &= \hspace{1cm} \frac{1}{\gamma} \Big(F_{yml} - \lambda_{ns}(Y_{bm} - Y_{bb})
  + \lambda_{nt}(Y_{t } - Y_{bm}) \Big) + R_{yml}
  &= \frac{\partial Y_{nm}}{\partial L_n}\dot{L}_n + \frac{\partial Y_{nm}}{\partial L_f}\dot{L}_f\\
  \dot{Y}_{t}  &= \hspace{1cm} \frac{1}{\gamma} \Big(F_{yt } - \lambda_{nt}(Y_{t } - Y_{bm})
  + \lambda_{ft}(Y_{fm} - Y_{t }) \Big) + R_{yt }
  &= \frac{\partial Y_{t}}{\partial L_n}\dot{L}_n + \frac{\partial Y_{t}}{\partial L_f}\dot{L}_f\\
  \dot{Y}_{fm} &= \hspace{1cm} \frac{1}{\gamma} \Big(F_{ymr} - \lambda_{ft}(Y_{fm} - Y_{t })
  + \lambda_{fs}(Y_{fb} - Y_{fm}) \Big) + R_{ymr}
  &= \frac{\partial Y_{fm}}{\partial L_n}\dot{L}_n + \frac{\partial Y_{fm}}{\partial L_f}\dot{L}_f
\end{align}

The matrix version of this system looks like:

\[
\begin{pmatrix}
  -\frac{X_{nm}}{Ln} & -\frac{X_{nm}}{Lf}
  & -\frac{X_{nm}-X_{nb}}{\gamma} & \frac{X_{t}-X_{nm}}{\gamma} & 0 & 0\\
  -\frac{X_{t}}{Ln} & -\frac{X_{t}}{Lf}
  & 0 & -\frac{X_{t}-X_{nm}}{\gamma} & \frac{X_{fm}-X_{t}}{\gamma} & 0\\
  -\frac{X_{fm}}{Ln} & -\frac{X_{fm}}{Lf}
  & 0 & 0 & -\frac{X_{fm}-X_{t}}{\gamma} & \frac{X_{fb}-X_{fm}}{\gamma}\\
  -\frac{Y_{nm}}{Ln} & -\frac{Y_{nm}}{Lf}
  & -\frac{Y_{nm}-Y_{nb}}{\gamma} & \frac{Y_{t}-Y_{nm}}{\gamma} & 0 & 0\\
  -\frac{Y_{t}}{Ln} & -\frac{Y_{t}}{Lf}
  & 0 & -\frac{Y_{t}-Y_{nm}}{\gamma} & \frac{Y_{fm}-Y_{t}}{\gamma} & 0\\
  -\frac{Y_{fm}}{Ln} & -\frac{Y_{fm}}{Lf}
  & 0 & 0 & -\frac{Y_{fm}-Y_{t}}{\gamma} & \frac{Y_{fb}-Y_{fm}}{\gamma}\\
\end{pmatrix}
\begin{pmatrix}
  \dot{L}_n\\
  \dot{L}_f\\
  \lambda_{ns}\\
  \lambda_{nt}\\
  \lambda_{ft}\\
  \lambda_{fs}\\
\end{pmatrix}
=
\begin{pmatrix}
  -\frac{1}{\gamma}F_{nmx} - R_{nmx}\\
  -\frac{1}{\gamma}F_{tx}  - R_{tx}\\
  -\frac{1}{\gamma}F_{fmx} - R_{fmx}\\
  -\frac{1}{\gamma}F_{nmy} - R_{nmy}\\
  -\frac{1}{\gamma}F_{ty}  - R_{ty}\\
  -\frac{1}{\gamma}F_{fmy} - R_{fmy}\\
\end{pmatrix}
\]

This system of equations is then solved and used to calculate $\dot{L}_n$ and $\dot{L}_f$, as shown in Appendix (\ref{sec:bb-motion-equations}).

\section{Data interpretation}
Metrics for quality are required to assess how well the model fits reality. These metrics include time spent in one/bothbound states and average step size. These metrics are developed here.\\

\subsection{Converting experimental rates to preexponential factors $A_b$ and $A_{ub}$}
Rate constants $k_b$ and $k_{ub}$ govern the probability of transitioning from onebound to bothbound and vice-versa, respectively. One issue is that when these parameters are measured experimentally, the values represent the average rate for an ensemble of dynein in many different conformations. In contrast, the rates the model uses represent the probability of unbinding of a specific conformation. Thus, it is necessary to find a way to convert between experimental rates $k_b^{exp}$ and $k_{ub}^{exp}$ and conformation-specific rates $k_b$ and $k_{ub}$. As shown in Section \ref{sec:transitioning-states}, rates can be expressed using the Arrhenius equation:

\begin{align*}
  k_b &= A_b'e^{\frac{-\Delta G_{bind}}{k_BT}} = A_b'e^{\frac{-\Delta G_{conf}^{ob\rightarrow bb}}{k_BT}}e^{\frac{-\Delta G_{bind}}{k_BT}} = A_be^{\frac{-\Delta G_{conf}^{ob\rightarrow bb}}{k_BT}}\\
  k_{ub} &= A_{ub}'e^{\frac{-\Delta G_{unbind}}{k_BT}} = A_{ub}'e^{\frac{-\Delta G_{conf}^{bb\rightarrow ob}}{k_BT}}e^{\frac{\Delta G_{bind}}{k_BT}} = A_{ub}e^{\frac{-\Delta G_{conf}^{bb\rightarrow ob}}{k_BT}}
\end{align*}

where the constant Boltzmann factor for binding energy are hidden inside the unprimed preexponential factors. The experimental values are calculated the same way, with the free energy changes replaced with expectation values. Since by the equipartition theorem $\left<G_{conf}^{ob}\right> = 2k_BT$ and $\left<G_{conf}^{bb}\right> = k_BT$, the equations can be simplified:

\begin{align*}
  k_b^{exp} &= A_be^{\frac{-\left(2k_BT - k_BT\right)}{k_BT}} = \frac{A_b}{e}\\
  k_{ub}^{exp} &= A_{ub}e^{\frac{-\left(k_BT-2k_BT\right)}{k_BT}} = A_{ub}e
\end{align*}

Thus experimental rates can be converted easily into the model parameters $A_b$ and $A_{ub}$.

\subsection{Literature values for experimental binding rates}
Sarlah \textit{et. al.} \cite{sarlahmodel} perform a similar simulation of dynein and provide the reaction rates shown in Figure \ref{fig:sarlah-rate-table}.

\begin{figure}[h]
  \centering
  \includegraphics[width=.45\textwidth]{../../figures/SarlahParameterTable.png}
  \caption{Sarlah kinetic rates.}{Kinetic rates used in Sarlah \textit{et. al.}'s model \cite{sarlahmodel}.}
  \label{fig:sarlah-rate-table}
\end{figure}

The assumption of this project is that many of the states in this table can be combined into a single state without loss of predictive power of dynein behavior. Specifically, the assumption is that ejecting phosphate, conducting the powerstroke, releasing ADP, binding ATP and dissociating from the microtubule and all occur during the bothbound state, and hydrolyzing ATP, conducting the prestroke and rebinding the MT occur during onebound. These states are roughly grouped as MT-bound and MT-unbound states in the Cianfrocco mechanochemical cycle \cite{cianfroccoreview}.\\

The rate-limiting transitions in the Cianfrocco model are used as the guess $k_b$ and $k_{ub}$ values. That means that, since ATP hydrolysis has the slowest rate constant in onebound and MT release has the slowest rate in bothbound, $k_b^{exp} = 180 s^{-1}$ and $k_{ub} = 460 s^{-1}$ serve as good initial guesses.\\

\subsection{Calculating one/bothbound times of experimental dynein}
Rate constants can be fit to experimentally measurable values, such as velocity and the time dynein is bound to the microtubule (processivity time). We define the following values:

\newcommand\tbb{\left<t_{bb}\right>}
\newcommand\tob{\left<t_{ob}\right>}
\newcommand\tstep{\left<t_{step}\right>}
\newcommand\tproc{\left<t_{processivity}\right>}
\newcommand\kub{k_{ub}}
\newcommand\kb{k_{b}}
\newcommand\ko{k_{dis}}

\begin{itemize}
\item $\tbb$: Time in bothbound per step
\item $\tob$: Time in onebound per step
\item $\tstep$: Time of a single step. $\tstep=\tbb+\tob$.
\item $P_{bb}$: Probability of being bothbound per unit time. $P_{bb} = \frac{\tbb}{\tstep}$
\item $P_{ob}$: Probability of being onebound per unit time. $P_{ob} = \frac{\tob}{\tstep}$
\item $\kub$: Rate of unbinding per unit time while in bothbound. $\kub = \tbb^{-1}$
\item $\kb$: Rate of binding per unit time while in onebound. $\kb = \tob^{-1}$
\item $\ko$: Rate of unbinding per unit time while in onebound. $\ko = P_{ob}\kub$
\item $\tproc$: Time bound to microtubule before dissociation. step length / velocity $= \ko^{-1}$
\end{itemize}

$\tbb$ and $\tob$ can thus be defined in terms of observables $\tstep$ and $\tproc$:

\begin{align*}
  \ko &= P_{ob}\kub = \frac{\tob}{\tstep\tbb}\\
  &= \frac{\tstep-\tbb}{\tstep\tbb}\\
  &= \frac{\tstep-\tbb}{\tstep\tbb}\\
  \tproc &= \frac{\tstep\tbb}{\tstep-\tbb}\\
  \tbb &= \frac{\tproc\tstep}{\tproc+\tstep}\\
  \tob &= \tstep-\tbb\\
  &= \tstep - \frac{\tstep\tproc}{\tstep+\tproc}\\
  &= \frac{\tstep^2}{\tstep+\tproc}\\
\end{align*}

Thus, by the above equations and using data from \cite{weihongpaper}, values for $\tbb$ and $\tob$ can be found:

\begin{figure}[h]
  \centering
  \begin{tabular}{| c | c | c | c | c | c |}
    \hline
    Velocity & $\left<L_{step}\right>$ & $\tstep$ & $\tproc$ & $\tbb$ & $\tob$ \\ \hline
    134nm/s & 8nm & 0.06s & 7.9s & 0.0595s & 4.52e-4\\ \hline
  \end{tabular}
  \caption{Experimental values for fitting.}{Table calculating estimates for $\tbb$ and $\tob$.}
  \label{table:time-parameter-table}
\end{figure}

\chapter{Methods}
\label{chap:Methods}
This section will go into detail on how the model was implemented in code, verified for accuracy, and how its parameters were fit to experiment.\\

\section{Simulation}
To generate stepping data, the models were coded in \verb|C++| and time-evolved using Euler's method to generate reasonable dynamical behavior. When in a position deemed worthy of a state transition a conversion, the model was converted to the respective other model, representing part of a ``step'' through the mechanochemical cycle. Some unphysical but minor corrections at points in the simulation were needed to rescue the model from entering incalculable positions.\\

Code is available at \verb|https://github.com/elliotc12/dynein_walk.git|.

%% \paragraph{Euler’s method}
%% Show how Euler's method is arrived at, how its error scales, why we use it.

%% \paragraph{Vs other methods (Runge Kutta, etc)}
%% Talk about why our differential equation is most conducive to Euler's method. Euler's is more
%% efficient -- is there a way to show how our differential equation is smooth, not jagged?

\textit{todo: come up with a better word than ``model'' to describe the state of the system at any time}
\subsection{Time evolution}
Euler's method for solving differential equations was used to calculate domain positions through time. Because each model is a set of multiple coupled differential equations, care was taken to update each domain position at the same time, and not in a cascade. A code snippet is shown in Figure (\ref{euler-code-snippet}) for the onebound updating scheme.

\begin{figure}[h!]
  \centering
  \begin{tabular}{c}
    \begin{lstlisting}[language=python]
      def simulate(dyn_ob):
      t = 0
      while (t < runtime):
      double temp_bba = dyn_ob->get_bba() + dyn_ob->get_d_bba() * dt
      double temp_bma = dyn_ob->get_bma() + dyn_ob->get_d_bma() * dt
      double temp_uma = dyn_ob->get_uma() + dyn_ob->get_d_uma() * dt
      double temp_uba = dyn_ob->get_uba() + dyn_ob->get_d_uba() * dt

      dyn_ob->set_bba(temp_bba)
      dyn_ob->set_bma(temp_bma)
      dyn_ob->set_uma(temp_uma)
      dyn_ob->set_uba(temp_uba)

      t += dt
    \end{lstlisting}
  \end{tabular}
  \caption{Code snippet}{Code snippet of Euler's method to update the positions of the onebound model simultaneously.}
  \label{ob_struct}
\end{figure}

The \verb|dt| used to update was $10^{-11}$ seconds.

\subsection{Transitioning between states}

\paragraph{Transition rate}
\label{sec:transitioning-states}
Transitions from one state to another are accomplished by calculating the probability of a transition at a given timestep, given by $P_{ob\rightarrow bb} = k_bdt$. Each timestep a value in $[0,1]$ is sampled from a random number generator and, if lower than $P$, the respective transition is made. A code snippet illustrating how unbinding occurs is shown in Figure (\ref{transition-rate-snippet}).

\begin{figure}[h!]
  \begin{tabular}{c}
    \begin{lstlisting}[language=python]
      def get_near_unbinding_rate(dyn_bb) {
        dyn_ob_test = make_onebound_from_bothbound(dyn_bb) // do a fake unbind to find energy
        dG_spring = dyn_ob_test.get_PE() - dyn_bb.get_PE()
        low_affinity_unbinding_preexponential_factor = low_affinity_unbinding_rate / exp(1.0)
        return low_affinity_unbinding_preexponential_factor*exp(-dG_spring/kb/T)
      }
      def transition_bothbound_to_onebound(dyn_bb) {
        random_number = rand(0,1)
        if (random_number < get_unbinding_rate(dyn_bb)) {
          return make_onebound_from_bothbound(dyn_bb)
        }
        else return 0
      }
    \end{lstlisting}
  \end{tabular}
  \caption{Code snip}{Code snippet of transition rate calculation using the Arrhenius equation.}
  \label{transition-rate-snippet}
\end{figure}

\paragraph{OB $\rightarrow$ BB Transition calculation}
When binding occurs, conversion between OB and BB models is done by creating a bothbound dynein which is as similar in terms of domain position as possible to the onebound dynein.\\

Due to the nature of the simulation, binding attempts must occur above, but very close to, the microtubule. This is because \textit{in vivo}, electrostatic interactions between MTBD and MT bring the two together when they are near. These electrostatic interactions are not present in the model, and so are simulated by allowing binding attempts to occur when $Y_{ub} \leq 0.2nm$ above the MT. When a binding attempt is successful, the unbound binding domain of the OB dynein is ``teleported'' such that $Y_{ub} = 0.0$. This is to enforce the constraint that both binding domains of the BB model are directly on the MT.\\

To create a BB model from OB, the following intermediate variables are first computed:\\

\begin{align}
  L_n^2 &= \left(X_t - X_{nb}\right)^2 + Y_t^2\\
  L_f^2 &= \left(X_t - X_{fb}\right)^2 + Y_t^2\\
  \cos(\Theta_{nm}) &= \frac{L_s^2 + L_t^2 - L_n^2}{2L_sL_t}\\
  \cos(\Theta_{fm}) &= \frac{L_s^2 + L_t^2 - L_f^2}{2L_sL_t}
\end{align}

\textbf{todo: figure out why this new method is used instead of just computing the bad nma angles}

\paragraph{BB $\rightarrow$ OB Transition calculation}
Similar to the binding transition, unbinding involves creating a new onebound dynein from the old bothbound. To accomplish a transition to the farbound state, the following equations are used:\\

\begin{align}
  \Theta_{new,bba} &= \Theta_{old,nba}\\
  \Theta_{new,bma} &= \Theta_{old,nma} + \Theta_{old,nba} - \pi\\
  \Theta_{new,uma} &= \Theta_{old,fma} + \Theta_{old,fba} - \pi\\
  \Theta_{new,uba} &= \Theta_{old,fba}
\end{align}

To do a nearbound transition, the ``f'' and ``n'' subscripts in the above equations are swapped.\\

As shown in Section \ref{final-parameter-table}, the MT binding rate is much quicker than the unbinding rate. This means that, when just unbound, the rate of rebinding is very high. To prevent immediate rebinding, the model is given a ``rebinding immunity'' of a single timestep. During this immunity period, rebinding is not allowed.\\

\subsection{Corrections}
\label{corrections-section}
Certain configurations of the model, such as in the bothbound model when $\Theta_{nm} = \pi$ or $\Theta_{fm} = \pi$, it becomes impossible to calculate intermediate variables due to values going to infinity (name for this??). For example when $\Theta_{nm} = \pi$, $Ln = Ls + Lt$ and so:

\begin{align}
  \cos\theta_{ns} &= 1 \\
  \sin\theta_{ns} &= 0 \\
  \mbox{some val?} &\rightarrow \infty
\end{align}

Alternative problem states are when $L = 0$ and ???. These states are corrected similarly.\\

\subsection{Forces}
Spring and Brownian forces are used to properly move the model through time. Appendices (\ref{sec:ob-motion-equatons}-\ref{sec:bb-motion-equations}) show angular velocities are functions of the x- and y-components of both spring forces $F$ and Brownian forces $\gamma R$.

\paragraph{Brownian forces}
Each Brownian force $R_{n,m}$ on domain $n$ in direction $m$ is sampled from a Gaussian with standard deviation $\sqrt{\frac{2k_BT\gamma_n}{dt}}$.\\

\paragraph{Spring forces}
Spring forces are calculated by first finding the torque $\tau$ on each angle, then converting this to a force. For example, the onebound near motor angle feels a force $\tau_{nm} = c_m(\Theta_{nm}-\Theta_{nm,eq})$, where $c_n$ is the spring constant of the motor domain and $\Theta_{nm,eq}$ the equilibrium angle. The magnitude of force on the adjacent binding and tail domains is thus $\tau/Ls$ and $\tau/Lt$, pointing in the direction adjacent the Ls and Lt rods, respectively. The force on the motor domain is found as the equal-and-opposite force to these two imposed forces. In code:

\begin{figure}[h]
  \centering
  \begin{tabular}{c}
    \begin{lstlisting}[language=C++]
      T = cm*(nma - nma_eq);
      f1 = T/Ls;
      f2 = T/Lt;
      f1x = f1 * sin(bba);
      f1y = f1 * -cos(bba);
      f2x = f2 * sin(bma);
      f2y = f2 * -cos(bma);
      f.bbx += f1x;
      f.bby += f1y;
      f.tx  += f2x;
      f.ty  += f2y;
      f.bmx += -(f1x + f2x); // equal and opposite forces
      f.bmy += -(f1y + f2y);
    \end{lstlisting}
  \end{tabular}
  \caption{Snip}{Code snippet of internal force calculation.}
  \label{transition-rate-snippet}
\end{figure}

\section{Verifying the model}
The models were verified to follow physical laws by checking their behavior in various tests: general conformational tests, energy conservation and obeying the equipartition theorem. The purpose of these tests was to verify that equations had been entered properly (primarily the conformational and energy conservation tests), and that the model obeyed the laws of physics (all tests).


\subsection{Conformational tests}
\label{sec:conformational-tests}
The models were tested in the following different conformations to verify they behaved as expected:\\

\textbf{Onebound tests}
\begin{itemize}
\item \textbf{Onebound:} \textit{Upwards line conformation with no forces.} Check that all domain x-values are zero$^*$ and all y-values are the proper addition of $L_s$ and $L_t$ lengths. Also checks that all angular velocities are zero. This test verifies that the angle scheme is defined properly, that the model can handle straight lines without generating NaNs and that there are no velocities (angular or cartesian) without forces.
\item \textbf{Onebound:} \textit{Horizontal line conformation with no forces.} Check that all domain y-values are zero and all x-values are the proper addition of $L_s$ and $L_t$ lengths. Also check that all velocities are zero. This test double-verifies that the coordinate system behaves as expected and that velocities are zero in the absence of forces.
\item \textbf{Onebound:} \textit{Upwards line conformation with +x forces} Check that cartesian x-velocities are positive and y-velocities are zero. Verifies that the cartesian velocities have the expected sign.
\item \textbf{Onebound:} \textit{Upwards line conformation with +y forces} Check that all cartesian velocities are zero. Verifies that upwards forces in an already-vertical stalk have no effect on velocity.
  \item \textbf{Onebound:} \textit{Prepowerstroke opposing internal/brownian forces} Check that, for two onebound models, one with +x spring forces and no Brownian forces and the other with -x Brownian forces and no spring forces, that the velocities of each unbound domain are equal and opposite. Verifies that equal Brownian and internal forces have the same magnitude and direction of effect on velocities.
\end{itemize}
*: checks for equality are within a margin of error of $\epsilon = 1e^{-5}$.\\

These tests together verify that the onebound model's internal coordinate system's directions match the XY coordinate system directions, that the angular directions in the model properly align with the $\pm x$ and $\pm y$ directions of the XY coordinate system, that velocities have the proper magnitude at zero force and proper sign at nonzero force, and that Brownian and internal forces are in the same unit system. There are still errors which could exist in the model pertaining to magnitude of velocity, which will be tested for in the energy conservation section.\\

\textbf{Bothbound tests}
\begin{itemize}
\item \textbf{Bothbound:} \textit{Upwards line conformation with no Brownian forces.} Both MTB domains are very close together ($L=10^(-25)$) and motor domains nearly vertical ($\Theta_{n/f} = \pi \pm 10^(-15)$). Equilibrium angles set to have a vertical line as the lowest energy point. Model is not exactly vertical for NaN concerns explained in section \ref{corrections-section}. Checks that domain x-values are zero$^*$, domain y-values are the correct combination of $L_s$ and $L_t$ lengths, all angles are at equilibrium and all forces are zero. Verifies that the domains are properly oriented by the angles, the angles are defined properly and forces are calculated from angles properly.
\item \textbf{Bothbound:} \textit{Equilateral house conformation with outward x-forces.} The model forms the shape of a square with an equilateral triangle on top, with the tail domain as the triangle. Checks if all domains are at the desired locations and the derivatives are the proper sign and are equal and opposite to their counterparts on the opposing side of the house. This test verifies that the angular system behaves as expected and forces are symmetric about the two dimers.
\item \textbf{Bothbound:} \textit{Two table conformations with near/far domains flipped.} Two models are put in the same table-like conformation, but the near/far domains are flipped. Checks if opposite domains have the same positions and forces. Re-verifies that the model is symmetric in position and force between the two dimers.
\end{itemize}

These tests verify the bothbound model's angle system behaves as expected, forces are zero when expected and have the proper signs, and that the whole system is symmetric with either dimer.

\subsection{Energy conservation}
\label{sec:energy-conservation}
To verify that the magnitudes of forces are calculated properly, a conservation of energy test was performed. A very slight angular nudge was applied to either model. This nudge caused a small change in energy. This change in energy was compared with the work the forces on each domain caused over the nudge. For energy to be conserved, these values were required to be very similar. Mathematically the test required the following:

\begin{equation}
  \sum_n \frac12 \left(\Theta_{nudge} - \Theta_{n} - \Theta_{n,eq}\right) = \sum_n \vec{F}_n \cdot \vec{r}_{n,nudge}
\end{equation}

where $\Theta_{nudge}$ was the angular nudge applied, $F_n$ the net force on domain $n$ and $\vec{r}_{n,nudge}$ the cartesian displacement of each domain $n$ due to the nudge. This test is powerful since it links the force-calculation system of either model to the energy calculation system. Energy is calculated by looking at displacements of angles from equilibrium, whereas forces are calculated using displacement of angles from equilibrium, but also orientation and position of each domain. Having the force system and energy system calculate the same results indicates that they are both likely correct.. \textbf{make this more precise.}

\textbf{is there a test to be run which could verify the velocity calculation is correct?}

\subsection{Equipartition theorem}
The equipartition theorem (ET) is a thermodynamic statement which says that all systems with solely quadratic energy dependence on $N$ degrees of freedom will have $<E> = \frac N2 k_bT$. For the dynein models to display ET behavior, they would have to both properly calculate forces and properly calculate velocities from forces. Brownian forces are also required to properly explore the different states of the system, but they need not be the correct magnitude for ET agreement.\\

To test if the models obey the ET, the total potential energy of each model was logged over \textbf{ten million} (???) timesteps of simulation. The results are reported in Figure \ref{equipartition_agreement}, where the PE scale is in units of $\frac12 k_bT$. Energies are reported as a running average over the simulation from 0 to time $t$. As shown, both models have their average potential energy converge to the expected amount, where the onebound model has four degrees of freedom and the bothbound two.\\

This test verifies both force and velocity are calculated properly for either model. If either force or velocity was wrong, then the model would not obey the Boltzmann hypothesis $P(s) = e^{-\Delta G_{s}/K_BT}$, where $s$ is a state of the system. The transition rate from $s_1$ to $s_2$ is $P(s_1\rightarrow s_2) = e^{-{\Delta G_2 - \Delta G_1}/K_BT}$. Wrong velocities are those that do not transition the system from $s_1$ to $s_2$ at this rate.\\

\textbf{would it be a stronger test to also calculate the standard deviation of <PE>? It seems like <PE> would still equal N/2KbT as long as the forces/velocities were symmetrically wrong. Calculating the stdev would require the state occupation to actually the ET. Or, maybe just plot the angle occupation and show that it is what it should be?}

\textbf{Possibly do mutations on motion equations to show that tests are no longer behaved}

\begin{figure}[h]
  \centering
  \begin{tabular}{| c | c | c | c |}
    \hline
    Author & Conditions & Velocity & Run length\\\hline
    Qiu \cite{weihongpaper} & 1mM [ATP] & $134 \pm 60.4$ nm & $1.06 \pm 0.044 \mu m$\\ \hline
    Reck-Peterson \cite{reck2006single} & 1mM [ATP] & $85 \pm 30 nm/s$  & $1.9 \pm 0.2 \mu m$\\ \hline
  \end{tabular}
  \caption{Literature values for experimental dynein stepping behavior.}{Table of literature values for dynein behavior}
  \label{table:parameter-table}
\end{figure}

%% \section{Data collection}
%% Time evolution of the system has the potential to create a huge amount of data, so care has to be taken to only collect what is necessary. For the purposes of testing stepping behavior of the model the geometry and per-domain forces and energies can be safely ignored. All that is needed to determine model fitness is the size and duration of each step.\\

%% \subsection{Stepping data output and calculation}
%% A step is defined as a complete cycle of 

%% \subsection{Simulation running via SLURM}

\section{Parameter fitting}
There are 14 free variables in the dynein model: rigid rod lengths $L_s$ and $L_t$, domain radii $R_b$, $R_m$ and $R_t$, spring constants $c_b$, $c_m$ and $c_t$, equilibrium angles $\theta_{b,eq}$, $\theta_{t,eq}$, $\Theta_{m,eq}^{Pre}$, $\Theta_{m,eq}^{Post}$, onebound binding rate $k_b$, and bothbound unbinding rate $k_{ub}$. Some of these values are found easily from literature or by looking at crystal structures of the motor protein, as shown in Table \ref{parameter-table}.\\

Others are more difficult to find. These values, including $k_b$, $k_ub$ and the three spring constants, are fit by an optimization process. First un/binding rates are fit to one/bothbound times, then spring constants are fit to average step length. This process is chronicled below.

\begin{figure}[h]
  \centering
  \begin{tabular}{| l | l | l | l | l | l | l | l | l |}
    \hline
    & Winch (Sarlah) & Schmidt & Lin & PyMol 3VKH & PyMol 4RH7 & Redwine & Kon & Burgess \\\hline
    $L_s$ & 12nm & & & 21.0nm & 22.1nm & & & \\ \hline
    $L_t$ &  7nm & & & & 11.15nm & & & \\ \hline
    $R_b$ &  & & & 1.57nm & 1.45nm & & & \\ \hline
    $R_m$ &  7nm & & & 7.36nm & 6.3nm & & & \\ \hline
    $R_t$ &  & & & & 2.16nm & & & \\ \hline
    $\theta_{m}^{Pre}$ & 250$^{\circ}$ & & 171$^{\circ}$ & & & & & 160\\ \hline
    $\theta_{m}^{Post}$ & 330$^{\circ}$ & & 137.5$^{\circ}$ & & & & & 136\\ \hline
    $\theta_{b}$ & 56$^{\circ}$ & & 63.5$^{\circ}$ & & & & & \\ \hline
    $\theta_{t}$ & 0$^{\circ}$ & & & & & & & \\ \hline
    $k_{ub}$ & $180 s^{-1,a}$ & & & & & & $90.2 \pm 4.5$& \\ \hline
    $k_b$ & $460 s^{-1,b}$ & & & & & & & \\ \hline
    $c_t$ & & & & & & & & \\ \hline
    $c_m$ & & & & & & & & \\ \hline
    $c_b$ & & & & & & 140 kJ/mol & & \\ \hline
  \end{tabular}
  \caption{Literature values for dynein model parameters.}{Table of literature values for various free parameters in dynein model.}
  \label{table:parameter-table}
\end{figure}

In addition, the physical conditions the simulations were run at are shown in Table \ref{table:simulation-conditions}. These conditions are meant to reflect the conditions found inside a typical cell. \textbf{of what species?}\\

\begin{figure}[h]
  \centering
  \begin{tabular}{| l | l |}
    \hline
    T & [ATP] \\ \hline
    310.15 K & $1mM$\\ \hline
  \end{tabular}
  \caption{Physical constants of simulation.}{Table of physical constants used in simulation.}
  \label{table:simulation-std-conditions}
\end{figure}

\subsection{Fitting rate constants}
To find the ideal binding and unbinding rates of our model, we ran simulations at various $k_b$ and $k_{ub}$ conditions. The time each simulation spent in each state, $t_{bb}$ and $t_{ob}$, was compared with the experimental data shown in Table \ref{table:time-parameter-table}. Simulation behavior for a wide range of $k_b$ and $k_{ub}$ values is shown as a heat map in Figure \ref{fig:kb-kub-heatmap}.\\

\textbf{Way to represent heatmap: show a blowup of the most promising section of the first heatmap, then a blowup of the second, etc until the optimum is found}

\subsection{Fitting spring constants}
This section will be written once it is conducted. This is contingent on getting rate fitting done, and preferably high-dt simulations stable so an optimization process can be run.\\

\chapter{Results \& Discussion}
\label{chap:ResultsAndDiscussion}

\section{Validating model}
The simulation was first validated for obeying the laws of physics, and in particular, the equipartition theorem. Figure \ref{fig:equipartition_agreement} shows how both models have their angular potential energy converge to the expected $\frac{n}{2}k_BT$ energy over a long time. This is evidence that both the forces and velocities of the model are properly calculated. Grossly inncorrect velocity or force calculations would lead to an incorrect occupation distrobution of angles, and hence an incorrect average energy. The model's force calculations are further validated in Section \ref{sec:conformational-tests}, and energy conservation is validated in Section \ref{sec:energy-conservation}.\\

\begin{figure}[h!]
  \centering
  \begin{minipage}[b]{0.45\textwidth}
    \includegraphics[width=\textwidth]{../../figures/OB_Average_PE.pdf}
    \caption{Onebound average spring energy.}{Average domain PE of onebound model converging to $\frac12 k_BT$ energy over time.}
  \end{minipage}
  \hspace{.2cm}
  \begin{minipage}[b]{0.45\textwidth}
    \includegraphics[width=\textwidth]{../../figures/BB_Average_PE.pdf}
    \caption{Bothbound average spring energy.}{Total domain PE of bothbound model converging to $\frac12 k_BT$ energy over time.}
  \end{minipage}
  \label{fig:equipartition_agreement}
\end{figure}

\section{Model achieves processivity}
When run at the proper parameters, the dynein model can step processively. Figure \ref{fig:stepping-trajectory} shows the dynamics of a motor when run at the parameters shown in Table \ref{table:thesis-parameters}.

\begin{figure}[h]
  \centering
  \begin{tabular}{| l | l | l | l | l | l |}
    \hline
    $k_{b}$ & $k_{ub}$ & $c_b$ &  $c_m$ & $c_t$ & dt\\\hline
    $10^{18} s^{-1}$ & $10^{20} s^{-1}$ & $2.0\Delta G_{ATP}$ & $2.0\Delta G_{ATP}$ & $1.0\Delta G_{ATP}$ & $10^{-10}s$\\ \hline
  \end{tabular}
  \caption{Parameters used in processive simulations in Figures \ref{fig:stepping-trajectory} and \ref{fig:final-histograms}.}
  \label{table:thesis-parameters}
\end{figure}

\begin{figure}[H]
  \centering
  \includegraphics[width=0.5\textwidth]{../../plots/trajectory-plot_thesis}
  \caption{Near and far binding domains over the course of several microseconds of simulation. Simulation run at standard conditions with $k_b=123 s^{-1}, k_{ub}=456 s^{-1}, c_b=c_m=c_t=789 \Delta G_{ATP}$.}
  \label{fig:stepping-trajectory}
\end{figure}

Figures \ref{fig:stepping-trajectory}.b and \ref{fig:stepping-trajectory}.c demonstrate the model stepping five times, where a step constitutes lifting a binding domain off the microtubule for any amount of time. This behavior indicates that a springy hinge model with shifting equilibrium angles is capable of generating processive stepping behavior. Now that the model has demonstrated stepping behavior, it must be seen if it can generate steps at a size- and time-scale similar to that which real dynein takes in experiment.\\

Another important determination from Figure \ref{fig:stepping-trajectory}.c is the average position of the unbound foot is about 30\textit{nm} above the microtubule. This is a preliminary finding which may not represent how the model will behave once fit to stepping behavior, so drawing conclusive conclusions from it is impossible. However, it does initially suggest how dynein searches for the microtubule. Instead of hovering near the microtubule looking for a binding site, the motor pulls its foot away from the microtubule, diffusing until pushed downward far enough to bind. This behavior could explain why dynein's stepping pattern is so large: diffusing downward from 30\textit{nm} would allow significant x-diffusion along the microtubule. This could be contrasted with kinesin, which is smaller and could not move its foot so high off the microtubule.\\

Importantly, the physical parameters causing processive motion are reasonable physically. The spring constants are roughly the same size as the energy of hydrolysis of ATP, which is reasonable. By a very rough argument, the dynein must move its tail angle by roughly one radian to diffuse enough for a step. This would correspond to a $\Delta G_{spring} = \frac{1}{2}c_t\Delta \theta_t \approx \Delta G_{ATP}$, which is very reasonable. In addition, the tail spring constant is less than the motor and binding spring constants, which is in line with the notion that the tail is much more elastic than the motor or binding domains. CITE\\

\section{Model step size is highly variant}
Figure \ref{fig:final-histograms}.a shows the stepping pattern of the model at the parameters shown in Table \ref{table:thesis-parameters}. This simulation was run over a much longer time period, and hence shows much more significant information on the model's stepping behavior.\\

\begin{figure}[H]
  \centering
    \includegraphics[width=0.5\textwidth]{../../plots/stepping_length_histogram_thesis}%%
    \includegraphics[width=0.5\textwidth]{../../figures/weihong-step-size.pdf}
    \caption{Model vs experimental step sizes. \textbf{a.)} Step sizes model took at parameters in \ref{table:thesis-parameters}. \textbf{b.)} Experimental stepping behavior of dynein \cite{weihongpaper}.}
    \label{fig:final-histograms}
\end{figure}

Figure \ref{fig:final-histograms}.a shows the model has a strong preference for $<5nm$ steps, but is capable of generating steps up to $25nm$ in length, with a fairly even distribution of step frequency between $5$ and $25nm$. This is a very important result, since it indicates that the model is capable of taking a varied step size. This behavior is characteristic of dynein, and hence very important to reproduce.\\

The preference for $<5nm$ steps is very un-dynein-like. One explanation is that steps may be defined differently between experimental and modeled dynein. In the simulation, every state transition where the motor unbinds from the microtubule is considered a step and plotted on the stepping histogram. This may not be how the experiment in \ref{fig:final-histograms}.b records steps. Another possibility is that the binding rate used for these simulations is far too high, and thus, immediate rebinding is very likely. A solution would thus be to lower the microtubule binding rate and see if the stepping histogram would level out. This may help, but may not, since lowering the binding rate would affect all binding probabilities, not just immediate rebindings.\\

The ability of the model to generate near-experiment stepping behavior is imperative if the model is to be considered a proper explanation of dynein's stepping. In the case that changing the binding rate does not remove the small-step preference of the model, it may be necessary to change how the model works. For example, using the total spring-energy-of-binding change to calculate the binding probability may be invalid, or perhaps the entire two-state model will need to be thrown out. The inability of either of these assumptions to predict dynein's behavior would themselves be important findings.\\

Another issue with the simulated histogram is that it shows no steps over $30nm$. This is a problem, since taking $40+nm$ steps is characteristic of dynein. Ideally, tuning the parameters of dynein would increase the step size. However, in the case that it can't, this might be an important result. The current model assumes connections between domains are rigid. This may not be a valid assumption, since the coiled coil region connecting the binding and motor domains has some elasticity to it CITE. Thus it may be necessary to add in elasticity to the model to reproduce long steps.\\

Comparing the simulation results in \ref{fig:final-histograms}.a with the experimental results in \ref{fig:final-histograms}.b, it is clear that the model is still far from experimental dynein. However, this is not indicative of a failure of the model, since the simulation parameters have not yet been tuned to dynein. It is very possible that changing the binding rate, unbinding rate and spring constants will change the stepping behavior of the model dramatically. Lowering the binding rate may even out the histogram and prevent instant rebinding. It may also allow the model to diffuse along the microtubule longer, generating longer steps than 25nm. Another issue is the lack of backwards steps, which may also be solved by altering the spring constants.\\

%% \section{Current work: Refining model parameters}
%% Once validated, the model was fit to experimental data. Figure \ref{fig:kb-kub-heatmap} shows a heat map of the behavior of simulations running at many different $k_b$ and $k_{ub}$ values. The color of each point represents the fit quality of a simulation's $\left<t_{ob}\right>$ value to experiment. As can be seen, a good fit is found in the $k_b=XX$, $k_{ub}=YY$ region.

%% \begin{figure}[H]
%%   \centering
%%   \includegraphics[width=\textwidth]{../../figures/kb-kub-contour}
%%   \caption{Heat map comparing $k_b$ and $k_{ub}$ values to time spent in the onebound state. Simulations run for XXXXX seconds at standard conditions shown in Table \ref{table:simulation-std-conditions}, with spring constants $c_b = c_m = c_t = 2.4\Delta G_{ATP}$, $L_s=22.1nm$ and $L_t=11.15nm$. The fit quality of each simulation is decided by $\log_{10}\left(\frac{\left<t_{bb}\right>_{sim}}{\left<t_{bb}\right>_{exp}}\right)$, then fit to the colorbar on the right. Values over or under the colorbar bounds are rounded to the max (1.0 or -1.0).}
%%   \label{fig:kb-kub-heatmap}
%% \end{figure}

\chapter{Conclusion}
\label{chap:Conclusion}

\section{Findings}
The most important findings of this work are that a springy hinge model of dynein can take steps of a similar size to real dynein. What's more, it is likely further tweaking of the model will improve the model's behavior to the point of generating backwards and large steps. This would be a very important result which would strongly support the linker-swing/diffusive-step model of dynein stepping and guide further research.\\

Many studies have been done on dynein, resulting in a large body of literature about the motor's behavior and features. Sifting through this information to find what is important for the motor's step is a difficult task. Mutational studies can remove traits of the motor, deducing what traits are necessary for stepping. However, this process is slow, and it can be difficult to remove large traits of the motor while keeping the engineered dynein representative of the natural motor. This is where simulations come in. Minimal features can be put into a physical model, which can then be tested for behavior similar to real dynein. In this study, we created a mathematical model which simplifies the motor to a rigid hinge structure with harmonic spring energies between the hinges. The equilibrium angles change dynamically to reflect structural changes during dynein's mechanochemical cycle. The model was then subjected to Brownian forces and evolved. If the model generates a stepping pattern similar to real dynein, this would demonstrate that dynein can be thought of as a rigid hinged structure. This would mean that many other traits of dynein may not be vital for stepping, such as stalk elasticity \cite{sarlahmodel}, motor-motor interaction \cite{sarlahmodel}, nucleotide state \cite{cianfroccoreview}, nucleotide state of non-AAA1 nuclease sites \cite{cianfroccoreview}, and changing stiffness of the foot domain \cite{burgess-paper}. Thus future research may want to focus more on linker swing and binding interactions, and less on these other traits.\\

Other findings cast doubt on some aspects of the model. The vast preference of the model for $<5nm$ steps suggests that the model may handle binding events unlike real dynein. When binding, the total spring energy of the transition from a onebound to a bothbound state is calculated, then used to calculate the likelihood of transition via a Boltzmann factor. This approach may be overly simple, or take into account the wrong information. It may be that it is necessary to expand the two-state model into a three-state model with a low-microtubule-affinity onebound state immediately after unbinding, to prevent instant rebinding. This state would eventually transition to the high-microtubule-affinity state after some time. If this proves to be necessary to decrease small steps, this would be an interesting result showing that the motor might rely on its low-microtule-affinity ADP-Pi state to prevent early binding.\\

\section{Further work}
The goal of studying dynein's mechanochemical cycle is to find an explanation for how it work. That is, to find the necessary and sufficient properties of dynein that cause the cycle to happen. Simulations are only part of the recipe for creating a scientific theory. Simulations are easy to implement and can study phenomena which are difficult to design experiments for. However, when experimental data is available, it is generally more reliable than a simulation of the same system. This is because a simulation is only as good as the strength of correspondence between the mathematical model and the system it represents. With dynein, this correspondence can be hard to achieve because the protein is complicated. The model predicts, according to the laws of physics for small objects, how a small machine composed of springy hinges would behave in an aqueous environment. This machine can be made the same size and general shape as dynein, and its conformational angles can be made the same as dynein's, and its state transitions can happen in the same way dynein's do. However, it is never possible to say that this machine ``is'' dynein or represents how dynein actually behaves. Thus, it is only possible to say that the simulation provides sufficient criteria for dynein stepping, not necessary criteria. The simulation shows that the linker-swing/diffusive-step model is sufficient for motion, but it cannot show that this is needed for dynein's walk.\\

To more strongly support the linker-swing/diffusive step model of dynein motion, more experiments should be done. Demonstrating that dynein requires flexible domain connections in order to take such broad steps would be one powerful experiment. Mutating the foot and tail domains to be more rigid, then examining stepping behavior would be one way to see if rigidity is necessary for random steps. Another experiment could use FRET (Forster Resonance Energy Transfer) and fluorophores on the microtubule and foot domain to track how the distance between the two regions evolves with time. This could provide information about the average distance between the MTBD and MT, which would validate this work's finding that the MTBD mostly hovers. More importantly, FRET data could be fit to models to predict the amount of diffusion the foot domain undergoes over the course of a step. Significant diffusion would indicate that diffusion may be very important for the step. Experiments with both feet labeled could get similarly good information.\\

\textbf{Talk to Dr. Hsu about this sort of thing...what experiments could be used to validate the model?}

If the model were validated by FRET and rigidity mutation studies, then there are many other interesting things which could be found from it. First, the amount of work done by the motor could be found by inducing an artificial drag force pulling the protein's motor domains against its direction of travel. This force could be integrated over dynein's trajectory to find the work the model exerted on its artificial cargo. This could be used to estimate the efficiency of the motor. Another interesting study would be to add binding sites to the model, only allowing it to bind at $\alpha-\beta$ tubulin dimerization sites, like real dynein. This would mandate re-fitting the binding constants to the new environment. It would be interesting to know if this model behaves more, or less dynein-like than the original. This would help answer the question of whether dynein searches for a binding site on the microtubule, or if it is naturally tuned to bind at the right site.\\

Another very interesting simulation which could be run could help answer the open question of why kinesin and dynein, though using similar mechanisms, produce such different steps. Kinesin, unlike dynein, conducts all of its chemistry directly on its foot. This translates to effectively having two fewer joints than dynein. Kinesin's stroke is believed to involves a similar foot unbinding, kicking forward, rebinding, then lurching the motor forward mechanism. The dynein model could be used to construct a very rough model of kinesin by shrinking the tail linkers and increasing the motor domain spring constant $c_m$ high enough to simulate a rigid hinge. This model could then be reduced to the size of kinesin, given the proper equilibrium angles, and simulated with Brownian dynamics. If this new model even roughly manages to mimic experimental kinesin stepping,that would be fascinating. This result would show that the differences betwen kinesin and dynein can be explained exclusively through the values of spring constants $c_t$, $c_m$, $c_b$, lengths $L_s$ and $L_t$, equilibrium angles and un/binding constants. This would mean the differences between the two motors are primarily due to the conformational angles the motors take with the microtubule and between their own domains, not because of differences in their mechanism.\\

Next: add Big Questions to introduction, then answer them here.\\

%----------------------------------------------------------------------------------------
%	THESIS CONTENT - APPENDICES
%----------------------------------------------------------------------------------------

\appendix % Cue to tell LaTeX that the following "chapters" are Appendices

% Include the appendices of the thesis as separate files from the Appendices folder
% Uncomment the lines as you write the Appendices

\chapter{Onebound equations} % Main appendix title
\label{sec:AppendixOneboundEquations}

The following is a continuation of the derivation for the full motion equations used to time evolve the onebound model:

\begin{align*}
AA &= L_s\sin\theta_{bb}      &   LL &= -L_s\cos(\theta_{bb})\\
BB &= L_t\sin(\theta_{bm})    &	 MM &= -L_t\cos(\theta_{bm})\\
CC &= -L_t\sin(\theta_{um})   &	 NN &= L_t\cos(\theta_{um})\\
DD &= -L_s\sin(\theta_{ub})   &	 OO &= L_s\cos(\theta_{ub})\\
\\
EE &= -\gamma_m (X_{bm} - X_{bb})       &  PP &= -\gamma_m (Y_{bm} - Y_{bb})\\
FF &= \gamma_m (X_{t } - X_{bm})        &	 QQ &= \gamma_m (Y_{t } - Y_{bm})\\ 
GG &= -\gamma_t (X_{t } - X_{bm})       &	 RR &= -\gamma_t (Y_{t } - Y_{bm})\\
HH &= \gamma_t (X_{um} - X_{t })        &	 SS &= \gamma_t (Y_{um} - Y_{t })\\ 
II &= -\gamma_m (X_{um} - X_{t })       &	 TT &= -\gamma_m (Y_{um} - Y_{t })\\
JJ &= \gamma_m (X_{ub} - X_{um})        &	 UU &= \gamma_m (Y_{ub} - Y_{um})\\ 
KK &= -\gamma_b (X_{ub} - X_{um})       &	 VV &= -\gamma_b (Y_{ub} - Y_{um})\\
\end{align*}%
%
\[
\begin{pmatrix}
  AA & 0 & 0 & 0 & EE & FF & 0 & 0\\
  AA & BB & 0 & 0 & 0 & GG & HH & 0\\
  AA & BB & CC & 0 & 0 & 0 & II & JJ\\
  AA & BB & CC & DD & 0 & 0 & 0 & KK\\
  LL & 0 & 0 & 0 & PP & QQ & 0 & 0\\
  LL & MM & 0 & 0 & 0 & RR & SS & 0\\
  LL & MM & NN & 0 & 0 & 0 & TT & UU\\
  LL & MM & NN & OO & 0 & 0 & 0 & VV\\
\end{pmatrix}
\begin{pmatrix}
    \dot{\theta}_{bb}\\
    \dot{\theta}_{bm}\\
    \dot{\theta}_{um}\\
    \dot{\theta}_{ub}\\
    \lambda_{bs}\\
    \lambda_{bt}\\
    \lambda_{ut}\\
    \lambda_{us}\\
  \end{pmatrix}
=
\begin{pmatrix}
  X1\\
  X2\\
  X3\\
  X4\\
  X5\\
  X6\\
  X7\\
  X8\\
\end{pmatrix}
\]

Solving the above system for $\dot{\Theta}_{bb}, \dot{\Theta}_{bm}, \dot{\Theta}_{um}$ and $\dot{\Theta}_{ub}$ gives the desired velocities. Solution was done via Mathematica.

\chapter{Bothbound equations} % Main appendix title
\label{sec:AppendixBothboundEquations}

The following are the full motion equations used to time evolve the bothbound model:

\begin{center}
  \begin{minipage}{0.3\textwidth}
    \begin{align*}
      a &= \frac{-dXnm}{dLn}\\
       b &= \frac{-dXnm}{dLf}\\
       c &= -(x_{nm} - x_{nb}) / \gamma_m\\
       d &= (x_t - x_{nm}) / \gamma_m\\
       e &= \frac{-dXt}{dLn}\\
       f &= \frac{-dXt}{dLf}\\
       g &= -(x_t - x_{nm}) / \gamma_t\\
       h &= (x_{fm} - x_t) / \gamma_t\\
       i &= \frac{-dXfm}{dLn}\\
       j &= \frac{-dXfm}{dLf}\\
       k &= -(x_{fm} - x_t) / \gamma_m\\
       l &= (x_{fb} - x_{fm}) / \gamma_m\\
    \end{align*}
  \end{minipage}
  \begin{minipage}{0.3\textwidth}
    \begin{align*}
       m &= \frac{-dYnm}{dLn}\\
       n &= \frac{-dYnm}{dLf}\\
       p &= -(y_{nm} - y_{nb}) / \gamma_m\\
       q &= (y_t - y_{nm}) / \gamma_m\\
       r &= \frac{-dYt}{dLn}\\
       s &= \frac{-dYt}{dLf}\\
       t &= -(y_t - y_{nm}) / \gamma_t\\
       u &= (y_{fm} - y_t) / \gamma_t\\
       v &= \frac{-dYfm}{dLn}\\
       w &= \frac{-dYfm}{dLf}\\
       x &= -(y_{fm} - y_t) / \gamma_m\\
       y &= (y_{fb} - y_{fm}) / \gamma_m
    \end{align*}
  \end{minipage}
\end{center}

Solving the above system for $\dot{\Theta}_{nm}$ and $\dot{\Theta}_{fm}$ gives the desired velocities. Solution was done via Mathematica.

%----------------------------------------------------------------------------------------
%	BIBLIOGRAPHY
%----------------------------------------------------------------------------------------

\printbibliography[heading=bibintoc]

%----------------------------------------------------------------------------------------

\end{document}  
